app/src/main/java/org/onionshare/android/ShareManager.kt

```diff
@@ -104,7 +104,13 @@ class ShareManager @Inject constructor(
         else if (!shouldStop && f is FilesState.Zipped &&
             (t is TorState.Stopping || t is TorState.Stopped || w is WebServerState.Stopped)
         ) {
-            emit(ShareUiState.Error(f.files))
+            notificationManager.onError()
+            val torFailed = (t as? TorState.Stopping)?.failedToConnect == true ||
+                (t as? TorState.Stopped)?.failedToConnect == true
+            LOG.info("Tor failed: $torFailed")
+            emit(ShareUiState.Error(f.files, torFailed))
+            // state hack to ensure the webserver also stops when tor fails, so we add files again
+            if (webserverManager.state.value !is WebServerState.Stopped) webserverManager.stop()
```

Comment:
I'm seeing a state-related crash on this branch that may be related to this change. I can reproduce the crash reliably on API 21 (Moto G 4G), but can't reproduce it on API 33 (Pixel 5).

* Choose a file
* Start sharing
* Wait for onion link to appear
* Stop sharing
* Wait for button to change back to "Start sharing"
* Start sharing
* Crash happens after a couple of seconds

```
---------------------------- PROCESS STARTED (11705) for package org.onionshare.android.nightly ----------------------------
13:59:01.258  W  Verification of void androidx.activity.ComponentActivity.<init>() took 303.110ms
13:59:01.678  D  Use EGL_SWAP_BEHAVIOR_PRESERVED: true
13:59:01.694  D  Validating map...
13:59:01.938  I  Rejecting re-init on previously-failed class java.lang.Class<androidx.compose.ui.platform.actionmodecallback.FloatingTextActionModeCallback>
13:59:01.938  I  Rejecting re-init on previously-failed class java.lang.Class<androidx.compose.ui.platform.actionmodecallback.FloatingTextActionModeCallback>
13:59:02.078  I  Background sticky concurrent mark sweep GC freed 7633(508KB) AllocSpace objects, 1(13KB) LOS objects, 2% free, 11MB/12MB, paused 37.899ms total 127.653ms
13:59:02.513  W  Android Keystore requires at least Android M
13:59:02.540  W  Android Keystore requires at least Android M
13:59:02.550  I  Provider GmsCore_OpenSSL not available
13:59:02.770  I  <qeglDrvAPI_eglInitialize:410>: EGL 1.4 QUALCOMM build: AU_LINUX_ANDROID_LA.BF.1.1.1_RB1.05.00.02.042.016_msm8226_LA.BF.1.1.1_RB1__release_AU ()
                 OpenGL ES Shader Compiler Version: E031.25.03.00
                 Build Date: 02/11/15 Wed
                 Local Branch: 
                 Remote Branch: quic/LA.BF.1.1.1_rb1.10
                 Local Patches: NONE
                 Reconstruct Branch: AU_LINUX_ANDROID_LA.BF.1.1.1_RB1.05.00.02.042.016 + 62ca4eb + acd831d + 9f8b442 + e027a02 + cba30ba + 53c303a + a649d79 + 23e16f8 + 5e97da7 + cbd2a44 + 33d072a + 7aacf06 + 72b33e7 + 28f6f60 + b4c13d8 +  NOTHING
13:59:02.775  I  Initialized EGL, version 1.4
13:59:02.867  D  Enabling debug mode 0
13:59:02.943  W  requestLayout() improperly called by androidx.compose.ui.platform.ViewLayerContainer{1597eae6 V.E..... ......I. 0,0-0,0} during layout: running second layout pass
13:59:02.944  D  Owner FocusChanged(true)
13:59:02.956  I  Skipped 67 frames!  The application may be doing too much work on its main thread.
13:59:03.379  W  requestLayout() improperly called by androidx.compose.ui.platform.ViewLayerContainer{1597eae6 V.E..... ......ID 0,0-0,0} during layout: running second layout pass
13:59:03.428  I  Background sticky concurrent mark sweep GC freed 72983(3MB) AllocSpace objects, 17(862KB) LOS objects, 16% free, 16MB/20MB, paused 4.567ms total 103.233ms
13:59:03.586  I  New state from: f-Added t-Stopped w-Stopped 
13:59:03.591  D  New state: NoFiles
13:59:15.821  D  endAllStagingAnimators on 0xb9050248 (UnprojectedRipple) with handle 0xb8fd4ce8
13:59:19.923  I  New state from: f-Added t-Stopped w-Stopped 
13:59:19.926  D  New state: FilesAdded
13:59:20.292  I  Background sticky concurrent mark sweep GC freed 58633(2MB) AllocSpace objects, 0(0B) LOS objects, 11% free, 17MB/20MB, paused 9.690ms total 75.653ms
13:59:22.137  I  New state from: f-Zipping t-Stopped w-Stopped 
13:59:22.137  E  Unhandled state: ↑
13:59:22.162  I  Starting...
13:59:22.174  I  New state from: f-Zipping t-Starting w-Stopped 
13:59:22.212  D  New state: Starting
---------------------------- PROCESS STARTED (12487) for package org.onionshare.android.nightly ----------------------------
13:59:22.423  W  libtor.so: unused DT entry: type 0x6ffffef5 arg 0x6b088
13:59:22.423  W  libtor.so: unused DT entry: type 0x6ffffffe arg 0x98504
13:59:22.423  W  libtor.so: unused DT entry: type 0x6fffffff arg 0x3
13:59:22.429  E  "libtor.so": ignoring 2-entry DT_PREINIT_ARRAY in shared library!
13:59:22.466  I  Acquired lock
13:59:22.604  I  New state from: f-Zipping t-Starting w-Stopped 
13:59:22.604  D  New state: Starting
13:59:22.711  D  TorService: Starting...
13:59:22.926  D  onStartCommand Intent { cmp=org.onionshare.android.nightly/org.onionshare.android.tor.ShareService }
13:59:22.940  I  New state from: f-Zipping t-Starting w-Stopped 
13:59:22.940  D  New state: Starting
13:59:22.948  I  New state from: f-Zipped t-Starting w-Stopped 
13:59:23.033  I  Autoreload is disabled because the development mode is off.
13:59:23.043  D  TorService: Starting...
13:59:23.142  I  Application started in 0.122 seconds.
13:59:23.142  I  Application started: io.ktor.server.application.Application@169d5c69
13:59:23.177  W  Suspending all threads took: 6.726ms
13:59:23.197  I  Background sticky concurrent mark sweep GC freed 32066(1098KB) AllocSpace objects, 0(0B) LOS objects, 4% free, 19MB/20MB, paused 11.606ms total 84.093ms
13:59:23.205  I  Rejecting re-init on previously-failed class java.lang.Class<io.netty.util.internal.LongAdderCounter>
13:59:23.205  I  Rejecting re-init on previously-failed class java.lang.Class<io.netty.util.internal.LongAdderCounter>
13:59:23.230  I  Your platform does not provide complete low-level API for accessing direct buffers reliably. Unless explicitly requested, heap buffer will always be preferred to avoid potential system instability.
13:59:23.238  E  No implementation found for int io.netty.channel.kqueue.Native.sizeofKEvent() (tried Java_io_netty_channel_kqueue_Native_sizeofKEvent and Java_io_netty_channel_kqueue_Native_sizeofKEvent__)
13:59:23.245  E  No implementation found for int io.netty.channel.epoll.Native.offsetofEpollData() (tried Java_io_netty_channel_epoll_Native_offsetofEpollData and Java_io_netty_channel_epoll_Native_offsetofEpollData__)
13:59:23.479  I  Responding at http://0.0.0.0:17638
13:59:23.482  I  New state from: f-Zipped t-Starting w-Started 
13:59:23.549  I  Background partial concurrent mark sweep GC freed 16754(766KB) AllocSpace objects, 1(20KB) LOS objects, 40% free, 19MB/32MB, paused 741us total 104.409ms
13:59:29.637  D  TorService: Started
13:59:29.640  I  New state from: f-Zipped t-Starting w-Started 
13:59:29.642  D  New state: Starting
13:59:29.867  D  CIRC: 2 LAUNCHED BUILD_FLAGS=NEED_CAPACITY PURPOSE=GENERAL TIME_CREATED=2023-03-17T13:59:29.680141
13:59:29.867  D  STATUS_CLIENT: NOTICE BOOTSTRAP PROGRESS=80 TAG=ap_conn SUMMARY="Connecting to a relay to build circuits"
13:59:29.871  E  Starting hidden service...
13:59:29.871  I  New state from: f-Zipped t-Starting w-Started 
13:59:29.871  D  New state: Starting
13:59:29.931  D  STATUS_CLIENT: NOTICE BOOTSTRAP PROGRESS=85 TAG=ap_conn_done SUMMARY="Connected to a relay to build circuits"
13:59:29.932  I  New state from: f-Zipped t-Starting w-Started 
13:59:29.932  D  New state: Starting
13:59:29.943  D  STATUS_CLIENT: NOTICE BOOTSTRAP PROGRESS=89 TAG=ap_handshake SUMMARY="Finishing handshake with a relay to build circuits"
13:59:29.943  I  New state from: f-Zipped t-Starting w-Started 
13:59:29.944  D  New state: Starting
13:59:29.944  I  New state from: f-Zipped t-Starting w-Started 
13:59:29.944  I  Starting check job
13:59:29.944  D  New state: Starting
13:59:29.945  I  Waiting for Tor to start...
13:59:30.029  D  STATUS_CLIENT: NOTICE BOOTSTRAP PROGRESS=90 TAG=ap_handshake_done SUMMARY="Handshake finished with a relay to build circuits"
13:59:30.030  D  STATUS_CLIENT: NOTICE BOOTSTRAP PROGRESS=95 TAG=circuit_create SUMMARY="Establishing a Tor circuit"
13:59:30.031  I  New state from: f-Zipped t-Starting w-Started 
13:59:30.031  D  New state: Starting
13:59:30.078  D  CIRC: 2 EXTENDED $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2 BUILD_FLAGS=NEED_CAPACITY PURPOSE=GENERAL TIME_CREATED=2023-03-17T13:59:29.680141
13:59:30.161  D  CIRC: 2 EXTENDED $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2,$58BC63142C881F88D08E071DEB63DBA09DE62141~Torrot01 BUILD_FLAGS=NEED_CAPACITY PURPOSE=GENERAL TIME_CREATED=2023-03-17T13:59:29.680141
13:59:30.293  D  CIRC: 2 EXTENDED $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2,$58BC63142C881F88D08E071DEB63DBA09DE62141~Torrot01,$429E9D69C1CC93B03E291EB22F1ED5FD18E63ECD~stuffy BUILD_FLAGS=NEED_CAPACITY PURPOSE=GENERAL TIME_CREATED=2023-03-17T13:59:29.680141
13:59:30.293  D  CIRC: 2 BUILT $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2,$58BC63142C881F88D08E071DEB63DBA09DE62141~Torrot01,$429E9D69C1CC93B03E291EB22F1ED5FD18E63ECD~stuffy BUILD_FLAGS=NEED_CAPACITY PURPOSE=GENERAL TIME_CREATED=2023-03-17T13:59:29.680141
13:59:30.293  D  STATUS_CLIENT: NOTICE BOOTSTRAP PROGRESS=100 TAG=done SUMMARY="Done"
13:59:30.294  D  STATUS_CLIENT: NOTICE CIRCUIT_ESTABLISHED
13:59:30.295  I  New state from: f-Zipped t-Starting w-Started 
13:59:30.295  D  New state: Starting
13:59:30.854  D  CIRC: 3 LAUNCHED BUILD_FLAGS=NEED_CAPACITY PURPOSE=GENERAL TIME_CREATED=2023-03-17T13:59:30.779808
13:59:30.898  D  CIRC: 3 EXTENDED $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2 BUILD_FLAGS=NEED_CAPACITY PURPOSE=GENERAL TIME_CREATED=2023-03-17T13:59:30.779808
13:59:31.881  D  CIRC: 3 EXTENDED $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2,$CA180620A33507B8233B90868EA5EED2A3426040~kottos BUILD_FLAGS=NEED_CAPACITY PURPOSE=GENERAL TIME_CREATED=2023-03-17T13:59:30.779808
13:59:31.882  D  HS_DESC: CREATED qgsg2fjxfibd462aaa3tzdbwtzf4uzx6gohp563oiyzbyh4fbcnev4ad UNKNOWN UNKNOWN 2qWj37YKHL3ch0U9oCKV0twGIVvGH82qUKlEnMjXq7s
13:59:31.882  D  HS_DESC: CREATED qgsg2fjxfibd462aaa3tzdbwtzf4uzx6gohp563oiyzbyh4fbcnev4ad UNKNOWN UNKNOWN fUlWBCtARnP0bDVp8sDIAzrewrK7KdR+A9qpof2KzXE
13:59:31.883  D  CIRC: 4 LAUNCHED BUILD_FLAGS=IS_INTERNAL,NEED_UPTIME PURPOSE=HS_SERVICE_INTRO HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:31.246367
13:59:31.883  D  CIRC: 5 LAUNCHED BUILD_FLAGS=IS_INTERNAL,NEED_UPTIME PURPOSE=HS_SERVICE_INTRO HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:31.308569
13:59:31.883  D  CIRC: 6 LAUNCHED BUILD_FLAGS=IS_INTERNAL,NEED_UPTIME PURPOSE=HS_SERVICE_INTRO HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:31.371535
13:59:31.883  D  CIRC: 7 LAUNCHED BUILD_FLAGS=IS_INTERNAL,NEED_UPTIME PURPOSE=HS_SERVICE_INTRO HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:31.446468
13:59:31.883  D  CIRC: 8 LAUNCHED BUILD_FLAGS=IS_INTERNAL,NEED_UPTIME PURPOSE=HS_SERVICE_INTRO HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:31.508480
13:59:31.883  D  CIRC: 9 LAUNCHED BUILD_FLAGS=IS_INTERNAL,NEED_UPTIME PURPOSE=HS_SERVICE_INTRO HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:31.569765
13:59:31.883  D  CIRC: 10 LAUNCHED BUILD_FLAGS=IS_INTERNAL,NEED_UPTIME PURPOSE=HS_SERVICE_INTRO HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:31.631326
13:59:31.883  D  CIRC: 11 LAUNCHED BUILD_FLAGS=IS_INTERNAL,NEED_UPTIME PURPOSE=HS_SERVICE_INTRO HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:31.693680
13:59:31.883  D  CIRC: 12 LAUNCHED BUILD_FLAGS=IS_INTERNAL,NEED_UPTIME PURPOSE=HS_SERVICE_INTRO HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:31.755728
13:59:31.883  D  CIRC: 13 LAUNCHED BUILD_FLAGS=IS_INTERNAL,NEED_UPTIME PURPOSE=HS_SERVICE_INTRO HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:31.817078
13:59:31.946  D  CIRC: 14 LAUNCHED BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY,NEED_UPTIME PURPOSE=HS_VANGUARDS TIME_CREATED=2023-03-17T13:59:31.881767
13:59:31.991  D  CIRC: 4 EXTENDED $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2 BUILD_FLAGS=IS_INTERNAL,NEED_UPTIME PURPOSE=HS_SERVICE_INTRO HS_STATE=HSSI_CONNECTING REND_QUERY=qgsg2fjxfibd462aaa3tzdbwtzf4uzx6gohp563oiyzbyh4fbcnev4ad TIME_CREATED=2023-03-17T13:59:31.246367
13:59:31.992  D  CIRC: 7 EXTENDED $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2 BUILD_FLAGS=IS_INTERNAL,NEED_UPTIME PURPOSE=HS_SERVICE_INTRO HS_STATE=HSSI_CONNECTING REND_QUERY=qgsg2fjxfibd462aaa3tzdbwtzf4uzx6gohp563oiyzbyh4fbcnev4ad TIME_CREATED=2023-03-17T13:59:31.446468
13:59:31.992  D  CIRC: 11 EXTENDED $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2 BUILD_FLAGS=IS_INTERNAL,NEED_UPTIME PURPOSE=HS_SERVICE_INTRO HS_STATE=HSSI_CONNECTING REND_QUERY=qgsg2fjxfibd462aaa3tzdbwtzf4uzx6gohp563oiyzbyh4fbcnev4ad TIME_CREATED=2023-03-17T13:59:31.693680
13:59:31.992  D  CIRC: 6 EXTENDED $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2 BUILD_FLAGS=IS_INTERNAL,NEED_UPTIME PURPOSE=HS_SERVICE_INTRO HS_STATE=HSSI_CONNECTING REND_QUERY=qgsg2fjxfibd462aaa3tzdbwtzf4uzx6gohp563oiyzbyh4fbcnev4ad TIME_CREATED=2023-03-17T13:59:31.371535
13:59:31.992  D  CIRC: 10 EXTENDED $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2 BUILD_FLAGS=IS_INTERNAL,NEED_UPTIME PURPOSE=HS_SERVICE_INTRO HS_STATE=HSSI_CONNECTING REND_QUERY=qgsg2fjxfibd462aaa3tzdbwtzf4uzx6gohp563oiyzbyh4fbcnev4ad TIME_CREATED=2023-03-17T13:59:31.631326
13:59:31.992  D  CIRC: 13 EXTENDED $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2 BUILD_FLAGS=IS_INTERNAL,NEED_UPTIME PURPOSE=HS_SERVICE_INTRO HS_STATE=HSSI_CONNECTING REND_QUERY=qgsg2fjxfibd462aaa3tzdbwtzf4uzx6gohp563oiyzbyh4fbcnev4ad TIME_CREATED=2023-03-17T13:59:31.817078
13:59:32.079  D  CIRC: 4 EXTENDED $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2,$4F683B5B4F9A69F06AD9E83131B4678D4F1DDF78~thepinginthenorth BUILD_FLAGS=IS_INTERNAL,NEED_UPTIME PURPOSE=HS_SERVICE_INTRO HS_STATE=HSSI_CONNECTING REND_QUERY=qgsg2fjxfibd462aaa3tzdbwtzf4uzx6gohp563oiyzbyh4fbcnev4ad TIME_CREATED=2023-03-17T13:59:31.246367
13:59:32.079  D  CIRC: 6 EXTENDED $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2,$4F683B5B4F9A69F06AD9E83131B4678D4F1DDF78~thepinginthenorth BUILD_FLAGS=IS_INTERNAL,NEED_UPTIME PURPOSE=HS_SERVICE_INTRO HS_STATE=HSSI_CONNECTING REND_QUERY=qgsg2fjxfibd462aaa3tzdbwtzf4uzx6gohp563oiyzbyh4fbcnev4ad TIME_CREATED=2023-03-17T13:59:31.371535
13:59:32.079  D  CIRC: 11 EXTENDED $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2,$82A4BD0933B59E5A6F94BEA01A8EF55ABC031725~operator BUILD_FLAGS=IS_INTERNAL,NEED_UPTIME PURPOSE=HS_SERVICE_INTRO HS_STATE=HSSI_CONNECTING REND_QUERY=qgsg2fjxfibd462aaa3tzdbwtzf4uzx6gohp563oiyzbyh4fbcnev4ad TIME_CREATED=2023-03-17T13:59:31.693680
13:59:32.079  D  CIRC: 13 EXTENDED $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2,$4F683B5B4F9A69F06AD9E83131B4678D4F1DDF78~thepinginthenorth BUILD_FLAGS=IS_INTERNAL,NEED_UPTIME PURPOSE=HS_SERVICE_INTRO HS_STATE=HSSI_CONNECTING REND_QUERY=qgsg2fjxfibd462aaa3tzdbwtzf4uzx6gohp563oiyzbyh4fbcnev4ad TIME_CREATED=2023-03-17T13:59:31.817078
13:59:32.085  D  CIRC: 10 EXTENDED $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2,$A511F574066CDC228C79ED366007644995136BD8~iWantedToCancel BUILD_FLAGS=IS_INTERNAL,NEED_UPTIME PURPOSE=HS_SERVICE_INTRO HS_STATE=HSSI_CONNECTING REND_QUERY=qgsg2fjxfibd462aaa3tzdbwtzf4uzx6gohp563oiyzbyh4fbcnev4ad TIME_CREATED=2023-03-17T13:59:31.631326
13:59:32.085  D  CIRC: 7 EXTENDED $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2,$A511F574066CDC228C79ED366007644995136BD8~iWantedToCancel BUILD_FLAGS=IS_INTERNAL,NEED_UPTIME PURPOSE=HS_SERVICE_INTRO HS_STATE=HSSI_CONNECTING REND_QUERY=qgsg2fjxfibd462aaa3tzdbwtzf4uzx6gohp563oiyzbyh4fbcnev4ad TIME_CREATED=2023-03-17T13:59:31.446468
13:59:32.086  D  CIRC: 3 EXTENDED $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2,$CA180620A33507B8233B90868EA5EED2A3426040~kottos,$00DCAEAE3E54C32809E7F7CC4BF2A6FC68FC552F~Quintex55 BUILD_FLAGS=NEED_CAPACITY PURPOSE=GENERAL TIME_CREATED=2023-03-17T13:59:30.779808
13:59:32.086  D  CIRC: 3 BUILT $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2,$CA180620A33507B8233B90868EA5EED2A3426040~kottos,$00DCAEAE3E54C32809E7F7CC4BF2A6FC68FC552F~Quintex55 BUILD_FLAGS=NEED_CAPACITY PURPOSE=GENERAL TIME_CREATED=2023-03-17T13:59:30.779808
13:59:32.121  D  CIRC: 11 EXTENDED $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2,$82A4BD0933B59E5A6F94BEA01A8EF55ABC031725~operator,$6DFEB41C04CCE846871338E85DD5ACF5CFB6C1DD~Onyx BUILD_FLAGS=IS_INTERNAL,NEED_UPTIME PURPOSE=HS_SERVICE_INTRO HS_STATE=HSSI_CONNECTING REND_QUERY=qgsg2fjxfibd462aaa3tzdbwtzf4uzx6gohp563oiyzbyh4fbcnev4ad TIME_CREATED=2023-03-17T13:59:31.693680
13:59:32.121  D  CIRC: 13 EXTENDED $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2,$4F683B5B4F9A69F06AD9E83131B4678D4F1DDF78~thepinginthenorth,$E8965A79FB2F335194141E8968755524840C44B6~Piratenpartei08 BUILD_FLAGS=IS_INTERNAL,NEED_UPTIME PURPOSE=HS_SERVICE_INTRO HS_STATE=HSSI_CONNECTING REND_QUERY=qgsg2fjxfibd462aaa3tzdbwtzf4uzx6gohp563oiyzbyh4fbcnev4ad TIME_CREATED=2023-03-17T13:59:31.817078
13:59:32.173  D  CIRC: 5 EXTENDED $02B2152129686AE0BE05F3D1460591DD71C4BB15~0x67726579 BUILD_FLAGS=IS_INTERNAL,NEED_UPTIME PURPOSE=HS_SERVICE_INTRO HS_STATE=HSSI_CONNECTING REND_QUERY=qgsg2fjxfibd462aaa3tzdbwtzf4uzx6gohp563oiyzbyh4fbcnev4ad TIME_CREATED=2023-03-17T13:59:31.308569
13:59:32.173  D  CIRC: 8 EXTENDED $02B2152129686AE0BE05F3D1460591DD71C4BB15~0x67726579 BUILD_FLAGS=IS_INTERNAL,NEED_UPTIME PURPOSE=HS_SERVICE_INTRO HS_STATE=HSSI_CONNECTING REND_QUERY=qgsg2fjxfibd462aaa3tzdbwtzf4uzx6gohp563oiyzbyh4fbcnev4ad TIME_CREATED=2023-03-17T13:59:31.508480
13:59:32.173  D  CIRC: 12 EXTENDED $02B2152129686AE0BE05F3D1460591DD71C4BB15~0x67726579 BUILD_FLAGS=IS_INTERNAL,NEED_UPTIME PURPOSE=HS_SERVICE_INTRO HS_STATE=HSSI_CONNECTING REND_QUERY=qgsg2fjxfibd462aaa3tzdbwtzf4uzx6gohp563oiyzbyh4fbcnev4ad TIME_CREATED=2023-03-17T13:59:31.755728
13:59:32.173  D  CIRC: 9 EXTENDED $02B2152129686AE0BE05F3D1460591DD71C4BB15~0x67726579 BUILD_FLAGS=IS_INTERNAL,NEED_UPTIME PURPOSE=HS_SERVICE_INTRO HS_STATE=HSSI_CONNECTING REND_QUERY=qgsg2fjxfibd462aaa3tzdbwtzf4uzx6gohp563oiyzbyh4fbcnev4ad TIME_CREATED=2023-03-17T13:59:31.569765
13:59:32.173  D  CIRC: 14 EXTENDED $02B2152129686AE0BE05F3D1460591DD71C4BB15~0x67726579 BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY,NEED_UPTIME PURPOSE=HS_VANGUARDS TIME_CREATED=2023-03-17T13:59:31.881767
13:59:32.192  D  CIRC: 4 EXTENDED $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2,$4F683B5B4F9A69F06AD9E83131B4678D4F1DDF78~thepinginthenorth,$9CC3E9C1F4196D4F11F2B815F8EA0B630A1FCC88~BigDaddy BUILD_FLAGS=IS_INTERNAL,NEED_UPTIME PURPOSE=HS_SERVICE_INTRO HS_STATE=HSSI_CONNECTING REND_QUERY=qgsg2fjxfibd462aaa3tzdbwtzf4uzx6gohp563oiyzbyh4fbcnev4ad TIME_CREATED=2023-03-17T13:59:31.246367
13:59:32.192  D  CIRC: 10 EXTENDED $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2,$A511F574066CDC228C79ED366007644995136BD8~iWantedToCancel,$68FBE72B0D411C314EC7F272CB5B3F14C2429246~TORminion BUILD_FLAGS=IS_INTERNAL,NEED_UPTIME PURPOSE=HS_SERVICE_INTRO HS_STATE=HSSI_CONNECTING REND_QUERY=qgsg2fjxfibd462aaa3tzdbwtzf4uzx6gohp563oiyzbyh4fbcnev4ad TIME_CREATED=2023-03-17T13:59:31.631326
13:59:32.193  D  CIRC: 13 EXTENDED $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2,$4F683B5B4F9A69F06AD9E83131B4678D4F1DDF78~thepinginthenorth,$E8965A79FB2F335194141E8968755524840C44B6~Piratenpartei08,$67F55733B91B85F578C2A962D0E0BA0900CBB185~1blu3DEicebeer01b BUILD_FLAGS=IS_INTERNAL,NEED_UPTIME PURPOSE=HS_SERVICE_INTRO HS_STATE=HSSI_CONNECTING REND_QUERY=qgsg2fjxfibd462aaa3tzdbwtzf4uzx6gohp563oiyzbyh4fbcnev4ad TIME_CREATED=2023-03-17T13:59:31.817078
13:59:32.194  D  CIRC: 13 BUILT $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2,$4F683B5B4F9A69F06AD9E83131B4678D4F1DDF78~thepinginthenorth,$E8965A79FB2F335194141E8968755524840C44B6~Piratenpartei08,$67F55733B91B85F578C2A962D0E0BA0900CBB185~1blu3DEicebeer01b BUILD_FLAGS=IS_INTERNAL,NEED_UPTIME PURPOSE=HS_SERVICE_INTRO HS_STATE=HSSI_CONNECTING REND_QUERY=qgsg2fjxfibd462aaa3tzdbwtzf4uzx6gohp563oiyzbyh4fbcnev4ad TIME_CREATED=2023-03-17T13:59:31.817078
13:59:32.195  D  CIRC: 7 EXTENDED $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2,$A511F574066CDC228C79ED366007644995136BD8~iWantedToCancel,$8D89ECA4C93928711A0BD6DB80F547A09AEF2D67~bauruine BUILD_FLAGS=IS_INTERNAL,NEED_UPTIME PURPOSE=HS_SERVICE_INTRO HS_STATE=HSSI_CONNECTING REND_QUERY=qgsg2fjxfibd462aaa3tzdbwtzf4uzx6gohp563oiyzbyh4fbcnev4ad TIME_CREATED=2023-03-17T13:59:31.446468
13:59:32.243  D  CIRC: 6 EXTENDED $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2,$4F683B5B4F9A69F06AD9E83131B4678D4F1DDF78~thepinginthenorth,$BD4843CCB9E69299F4B47CF4322A864116B9B32C~BloodyTorRelay BUILD_FLAGS=IS_INTERNAL,NEED_UPTIME PURPOSE=HS_SERVICE_INTRO HS_STATE=HSSI_CONNECTING REND_QUERY=qgsg2fjxfibd462aaa3tzdbwtzf4uzx6gohp563oiyzbyh4fbcnev4ad TIME_CREATED=2023-03-17T13:59:31.371535
13:59:32.243  D  CIRC: 11 EXTENDED $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2,$82A4BD0933B59E5A6F94BEA01A8EF55ABC031725~operator,$6DFEB41C04CCE846871338E85DD5ACF5CFB6C1DD~Onyx,$8708EF197D06CDB65CF198233DB911F2F2008BAF~retard BUILD_FLAGS=IS_INTERNAL,NEED_UPTIME PURPOSE=HS_SERVICE_INTRO HS_STATE=HSSI_CONNECTING REND_QUERY=qgsg2fjxfibd462aaa3tzdbwtzf4uzx6gohp563oiyzbyh4fbcnev4ad TIME_CREATED=2023-03-17T13:59:31.693680
13:59:32.243  D  CIRC: 11 BUILT $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2,$82A4BD0933B59E5A6F94BEA01A8EF55ABC031725~operator,$6DFEB41C04CCE846871338E85DD5ACF5CFB6C1DD~Onyx,$8708EF197D06CDB65CF198233DB911F2F2008BAF~retard BUILD_FLAGS=IS_INTERNAL,NEED_UPTIME PURPOSE=HS_SERVICE_INTRO HS_STATE=HSSI_CONNECTING REND_QUERY=qgsg2fjxfibd462aaa3tzdbwtzf4uzx6gohp563oiyzbyh4fbcnev4ad TIME_CREATED=2023-03-17T13:59:31.693680
13:59:32.253  D  CIRC: 5 EXTENDED $02B2152129686AE0BE05F3D1460591DD71C4BB15~0x67726579,$4F683B5B4F9A69F06AD9E83131B4678D4F1DDF78~thepinginthenorth BUILD_FLAGS=IS_INTERNAL,NEED_UPTIME PURPOSE=HS_SERVICE_INTRO HS_STATE=HSSI_CONNECTING REND_QUERY=qgsg2fjxfibd462aaa3tzdbwtzf4uzx6gohp563oiyzbyh4fbcnev4ad TIME_CREATED=2023-03-17T13:59:31.308569
13:59:32.253  D  CIRC: 9 EXTENDED $02B2152129686AE0BE05F3D1460591DD71C4BB15~0x67726579,$4F683B5B4F9A69F06AD9E83131B4678D4F1DDF78~thepinginthenorth BUILD_FLAGS=IS_INTERNAL,NEED_UPTIME PURPOSE=HS_SERVICE_INTRO HS_STATE=HSSI_CONNECTING REND_QUERY=qgsg2fjxfibd462aaa3tzdbwtzf4uzx6gohp563oiyzbyh4fbcnev4ad TIME_CREATED=2023-03-17T13:59:31.569765
13:59:32.253  D  CIRC: 12 EXTENDED $02B2152129686AE0BE05F3D1460591DD71C4BB15~0x67726579,$4F683B5B4F9A69F06AD9E83131B4678D4F1DDF78~thepinginthenorth BUILD_FLAGS=IS_INTERNAL,NEED_UPTIME PURPOSE=HS_SERVICE_INTRO HS_STATE=HSSI_CONNECTING REND_QUERY=qgsg2fjxfibd462aaa3tzdbwtzf4uzx6gohp563oiyzbyh4fbcnev4ad TIME_CREATED=2023-03-17T13:59:31.755728
13:59:32.253  D  CIRC: 14 EXTENDED $02B2152129686AE0BE05F3D1460591DD71C4BB15~0x67726579,$4F683B5B4F9A69F06AD9E83131B4678D4F1DDF78~thepinginthenorth BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY,NEED_UPTIME PURPOSE=HS_VANGUARDS TIME_CREATED=2023-03-17T13:59:31.881767
13:59:32.254  D  CIRC: 8 EXTENDED $02B2152129686AE0BE05F3D1460591DD71C4BB15~0x67726579,$C237392DC121DF50C64C707C3FB95055E68B00CA~pacoCalienteRelay2 BUILD_FLAGS=IS_INTERNAL,NEED_UPTIME PURPOSE=HS_SERVICE_INTRO HS_STATE=HSSI_CONNECTING REND_QUERY=qgsg2fjxfibd462aaa3tzdbwtzf4uzx6gohp563oiyzbyh4fbcnev4ad TIME_CREATED=2023-03-17T13:59:31.508480
13:59:32.305  D  CIRC: 14 EXTENDED $02B2152129686AE0BE05F3D1460591DD71C4BB15~0x67726579,$4F683B5B4F9A69F06AD9E83131B4678D4F1DDF78~thepinginthenorth,$82A4BD0933B59E5A6F94BEA01A8EF55ABC031725~operator BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY,NEED_UPTIME PURPOSE=HS_VANGUARDS TIME_CREATED=2023-03-17T13:59:31.881767
13:59:32.305  D  CIRC: 14 BUILT $02B2152129686AE0BE05F3D1460591DD71C4BB15~0x67726579,$4F683B5B4F9A69F06AD9E83131B4678D4F1DDF78~thepinginthenorth,$82A4BD0933B59E5A6F94BEA01A8EF55ABC031725~operator BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY,NEED_UPTIME PURPOSE=HS_VANGUARDS TIME_CREATED=2023-03-17T13:59:31.881767
13:59:32.319  D  CIRC: 4 EXTENDED $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2,$4F683B5B4F9A69F06AD9E83131B4678D4F1DDF78~thepinginthenorth,$9CC3E9C1F4196D4F11F2B815F8EA0B630A1FCC88~BigDaddy,$67A54EDB7DADDBBC1E8A07F1EF82364BDAA896B7~fuckfbi BUILD_FLAGS=IS_INTERNAL,NEED_UPTIME PURPOSE=HS_SERVICE_INTRO HS_STATE=HSSI_CONNECTING REND_QUERY=qgsg2fjxfibd462aaa3tzdbwtzf4uzx6gohp563oiyzbyh4fbcnev4ad TIME_CREATED=2023-03-17T13:59:31.246367
13:59:32.320  D  CIRC: 4 BUILT $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2,$4F683B5B4F9A69F06AD9E83131B4678D4F1DDF78~thepinginthenorth,$9CC3E9C1F4196D4F11F2B815F8EA0B630A1FCC88~BigDaddy,$67A54EDB7DADDBBC1E8A07F1EF82364BDAA896B7~fuckfbi BUILD_FLAGS=IS_INTERNAL,NEED_UPTIME PURPOSE=HS_SERVICE_INTRO HS_STATE=HSSI_CONNECTING REND_QUERY=qgsg2fjxfibd462aaa3tzdbwtzf4uzx6gohp563oiyzbyh4fbcnev4ad TIME_CREATED=2023-03-17T13:59:31.246367
13:59:32.345  D  CIRC: 10 EXTENDED $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2,$A511F574066CDC228C79ED366007644995136BD8~iWantedToCancel,$68FBE72B0D411C314EC7F272CB5B3F14C2429246~TORminion,$7F0B0ACEEF829710E51CC3EE3A2A0E0C0993255B~bauruine BUILD_FLAGS=IS_INTERNAL,NEED_UPTIME PURPOSE=HS_SERVICE_INTRO HS_STATE=HSSI_CONNECTING REND_QUERY=qgsg2fjxfibd462aaa3tzdbwtzf4uzx6gohp563oiyzbyh4fbcnev4ad TIME_CREATED=2023-03-17T13:59:31.631326
13:59:32.345  D  CIRC: 10 BUILT $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2,$A511F574066CDC228C79ED366007644995136BD8~iWantedToCancel,$68FBE72B0D411C314EC7F272CB5B3F14C2429246~TORminion,$7F0B0ACEEF829710E51CC3EE3A2A0E0C0993255B~bauruine BUILD_FLAGS=IS_INTERNAL,NEED_UPTIME PURPOSE=HS_SERVICE_INTRO HS_STATE=HSSI_CONNECTING REND_QUERY=qgsg2fjxfibd462aaa3tzdbwtzf4uzx6gohp563oiyzbyh4fbcnev4ad TIME_CREATED=2023-03-17T13:59:31.631326
13:59:32.397  D  CIRC: 9 EXTENDED $02B2152129686AE0BE05F3D1460591DD71C4BB15~0x67726579,$4F683B5B4F9A69F06AD9E83131B4678D4F1DDF78~thepinginthenorth,$EA670AB58CD5E1D5DCAB571D100EA1D36D72CA28~billgates BUILD_FLAGS=IS_INTERNAL,NEED_UPTIME PURPOSE=HS_SERVICE_INTRO HS_STATE=HSSI_CONNECTING REND_QUERY=qgsg2fjxfibd462aaa3tzdbwtzf4uzx6gohp563oiyzbyh4fbcnev4ad TIME_CREATED=2023-03-17T13:59:31.569765
13:59:32.397  D  CIRC: 12 EXTENDED $02B2152129686AE0BE05F3D1460591DD71C4BB15~0x67726579,$4F683B5B4F9A69F06AD9E83131B4678D4F1DDF78~thepinginthenorth,$CDEAB87D26FC60F35474FEED676EBB13237D9FBC~Sparker BUILD_FLAGS=IS_INTERNAL,NEED_UPTIME PURPOSE=HS_SERVICE_INTRO HS_STATE=HSSI_CONNECTING REND_QUERY=qgsg2fjxfibd462aaa3tzdbwtzf4uzx6gohp563oiyzbyh4fbcnev4ad TIME_CREATED=2023-03-17T13:59:31.755728
13:59:32.399  D  CIRC: 6 EXTENDED $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2,$4F683B5B4F9A69F06AD9E83131B4678D4F1DDF78~thepinginthenorth,$BD4843CCB9E69299F4B47CF4322A864116B9B32C~BloodyTorRelay,$2CE96A8A1DA032664C90F574AFFBECE18A6E8DFC~Doedel24 BUILD_FLAGS=IS_INTERNAL,NEED_UPTIME PURPOSE=HS_SERVICE_INTRO HS_STATE=HSSI_CONNECTING REND_QUERY=qgsg2fjxfibd462aaa3tzdbwtzf4uzx6gohp563oiyzbyh4fbcnev4ad TIME_CREATED=2023-03-17T13:59:31.371535
13:59:32.399  D  CIRC: 6 BUILT $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2,$4F683B5B4F9A69F06AD9E83131B4678D4F1DDF78~thepinginthenorth,$BD4843CCB9E69299F4B47CF4322A864116B9B32C~BloodyTorRelay,$2CE96A8A1DA032664C90F574AFFBECE18A6E8DFC~Doedel24 BUILD_FLAGS=IS_INTERNAL,NEED_UPTIME PURPOSE=HS_SERVICE_INTRO HS_STATE=HSSI_CONNECTING REND_QUERY=qgsg2fjxfibd462aaa3tzdbwtzf4uzx6gohp563oiyzbyh4fbcnev4ad TIME_CREATED=2023-03-17T13:59:31.371535
13:59:32.399  D  CIRC: 7 EXTENDED $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2,$A511F574066CDC228C79ED366007644995136BD8~iWantedToCancel,$8D89ECA4C93928711A0BD6DB80F547A09AEF2D67~bauruine,$D5E5C2C54CD36E65AA5903E89E02148674A8C423~SubjectDelta BUILD_FLAGS=IS_INTERNAL,NEED_UPTIME PURPOSE=HS_SERVICE_INTRO HS_STATE=HSSI_CONNECTING REND_QUERY=qgsg2fjxfibd462aaa3tzdbwtzf4uzx6gohp563oiyzbyh4fbcnev4ad TIME_CREATED=2023-03-17T13:59:31.446468
13:59:32.399  D  CIRC: 7 BUILT $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2,$A511F574066CDC228C79ED366007644995136BD8~iWantedToCancel,$8D89ECA4C93928711A0BD6DB80F547A09AEF2D67~bauruine,$D5E5C2C54CD36E65AA5903E89E02148674A8C423~SubjectDelta BUILD_FLAGS=IS_INTERNAL,NEED_UPTIME PURPOSE=HS_SERVICE_INTRO HS_STATE=HSSI_CONNECTING REND_QUERY=qgsg2fjxfibd462aaa3tzdbwtzf4uzx6gohp563oiyzbyh4fbcnev4ad TIME_CREATED=2023-03-17T13:59:31.446468
13:59:32.408  D  CIRC: 5 EXTENDED $02B2152129686AE0BE05F3D1460591DD71C4BB15~0x67726579,$4F683B5B4F9A69F06AD9E83131B4678D4F1DDF78~thepinginthenorth,$8885EA6F74A694825B13B8A7080F6CF164DF74FB~Unnamed BUILD_FLAGS=IS_INTERNAL,NEED_UPTIME PURPOSE=HS_SERVICE_INTRO HS_STATE=HSSI_CONNECTING REND_QUERY=qgsg2fjxfibd462aaa3tzdbwtzf4uzx6gohp563oiyzbyh4fbcnev4ad TIME_CREATED=2023-03-17T13:59:31.308569
13:59:32.483  D  CIRC: 8 EXTENDED $02B2152129686AE0BE05F3D1460591DD71C4BB15~0x67726579,$C237392DC121DF50C64C707C3FB95055E68B00CA~pacoCalienteRelay2,$EC167B8C9A38A0F5F79D4A01479035760E0D3D63~zensursula BUILD_FLAGS=IS_INTERNAL,NEED_UPTIME PURPOSE=HS_SERVICE_INTRO HS_STATE=HSSI_CONNECTING REND_QUERY=qgsg2fjxfibd462aaa3tzdbwtzf4uzx6gohp563oiyzbyh4fbcnev4ad TIME_CREATED=2023-03-17T13:59:31.508480
13:59:32.531  D  CIRC: 12 EXTENDED $02B2152129686AE0BE05F3D1460591DD71C4BB15~0x67726579,$4F683B5B4F9A69F06AD9E83131B4678D4F1DDF78~thepinginthenorth,$CDEAB87D26FC60F35474FEED676EBB13237D9FBC~Sparker,$721A251658BA2240EED110C078334AB2DF6C309C~as212520 BUILD_FLAGS=IS_INTERNAL,NEED_UPTIME PURPOSE=HS_SERVICE_INTRO HS_STATE=HSSI_CONNECTING REND_QUERY=qgsg2fjxfibd462aaa3tzdbwtzf4uzx6gohp563oiyzbyh4fbcnev4ad TIME_CREATED=2023-03-17T13:59:31.755728
13:59:32.531  D  CIRC: 12 BUILT $02B2152129686AE0BE05F3D1460591DD71C4BB15~0x67726579,$4F683B5B4F9A69F06AD9E83131B4678D4F1DDF78~thepinginthenorth,$CDEAB87D26FC60F35474FEED676EBB13237D9FBC~Sparker,$721A251658BA2240EED110C078334AB2DF6C309C~as212520 BUILD_FLAGS=IS_INTERNAL,NEED_UPTIME PURPOSE=HS_SERVICE_INTRO HS_STATE=HSSI_CONNECTING REND_QUERY=qgsg2fjxfibd462aaa3tzdbwtzf4uzx6gohp563oiyzbyh4fbcnev4ad TIME_CREATED=2023-03-17T13:59:31.755728
13:59:32.531  D  CIRC: 12 BUILT $02B2152129686AE0BE05F3D1460591DD71C4BB15~0x67726579,$4F683B5B4F9A69F06AD9E83131B4678D4F1DDF78~thepinginthenorth,$CDEAB87D26FC60F35474FEED676EBB13237D9FBC~Sparker,$721A251658BA2240EED110C078334AB2DF6C309C~as212520 BUILD_FLAGS=IS_INTERNAL,NEED_UPTIME PURPOSE=HS_VANGUARDS TIME_CREATED=2023-03-17T13:59:31.755728
13:59:32.598  D  CIRC: 9 EXTENDED $02B2152129686AE0BE05F3D1460591DD71C4BB15~0x67726579,$4F683B5B4F9A69F06AD9E83131B4678D4F1DDF78~thepinginthenorth,$EA670AB58CD5E1D5DCAB571D100EA1D36D72CA28~billgates,$80A36AC3C045B1896DE853BB3AC652B8541FA0BD~cherryberries06 BUILD_FLAGS=IS_INTERNAL,NEED_UPTIME PURPOSE=HS_SERVICE_INTRO HS_STATE=HSSI_CONNECTING REND_QUERY=qgsg2fjxfibd462aaa3tzdbwtzf4uzx6gohp563oiyzbyh4fbcnev4ad TIME_CREATED=2023-03-17T13:59:31.569765
13:59:32.598  D  CIRC: 9 BUILT $02B2152129686AE0BE05F3D1460591DD71C4BB15~0x67726579,$4F683B5B4F9A69F06AD9E83131B4678D4F1DDF78~thepinginthenorth,$EA670AB58CD5E1D5DCAB571D100EA1D36D72CA28~billgates,$80A36AC3C045B1896DE853BB3AC652B8541FA0BD~cherryberries06 BUILD_FLAGS=IS_INTERNAL,NEED_UPTIME PURPOSE=HS_SERVICE_INTRO HS_STATE=HSSI_CONNECTING REND_QUERY=qgsg2fjxfibd462aaa3tzdbwtzf4uzx6gohp563oiyzbyh4fbcnev4ad TIME_CREATED=2023-03-17T13:59:31.569765
13:59:32.598  D  CIRC: 9 BUILT $02B2152129686AE0BE05F3D1460591DD71C4BB15~0x67726579,$4F683B5B4F9A69F06AD9E83131B4678D4F1DDF78~thepinginthenorth,$EA670AB58CD5E1D5DCAB571D100EA1D36D72CA28~billgates,$80A36AC3C045B1896DE853BB3AC652B8541FA0BD~cherryberries06 BUILD_FLAGS=IS_INTERNAL,NEED_UPTIME PURPOSE=HS_VANGUARDS TIME_CREATED=2023-03-17T13:59:31.569765
13:59:32.641  D  CIRC: 8 EXTENDED $02B2152129686AE0BE05F3D1460591DD71C4BB15~0x67726579,$C237392DC121DF50C64C707C3FB95055E68B00CA~pacoCalienteRelay2,$EC167B8C9A38A0F5F79D4A01479035760E0D3D63~zensursula,$6118DDB580EEDD6FE5F8D3CC0DF1EE17D8A2A4C6~DevilHunter BUILD_FLAGS=IS_INTERNAL,NEED_UPTIME PURPOSE=HS_SERVICE_INTRO HS_STATE=HSSI_CONNECTING REND_QUERY=qgsg2fjxfibd462aaa3tzdbwtzf4uzx6gohp563oiyzbyh4fbcnev4ad TIME_CREATED=2023-03-17T13:59:31.508480
13:59:32.641  D  CIRC: 8 BUILT $02B2152129686AE0BE05F3D1460591DD71C4BB15~0x67726579,$C237392DC121DF50C64C707C3FB95055E68B00CA~pacoCalienteRelay2,$EC167B8C9A38A0F5F79D4A01479035760E0D3D63~zensursula,$6118DDB580EEDD6FE5F8D3CC0DF1EE17D8A2A4C6~DevilHunter BUILD_FLAGS=IS_INTERNAL,NEED_UPTIME PURPOSE=HS_SERVICE_INTRO HS_STATE=HSSI_CONNECTING REND_QUERY=qgsg2fjxfibd462aaa3tzdbwtzf4uzx6gohp563oiyzbyh4fbcnev4ad TIME_CREATED=2023-03-17T13:59:31.508480
13:59:32.643  D  CIRC: 8 BUILT $02B2152129686AE0BE05F3D1460591DD71C4BB15~0x67726579,$C237392DC121DF50C64C707C3FB95055E68B00CA~pacoCalienteRelay2,$EC167B8C9A38A0F5F79D4A01479035760E0D3D63~zensursula,$6118DDB580EEDD6FE5F8D3CC0DF1EE17D8A2A4C6~DevilHunter BUILD_FLAGS=IS_INTERNAL,NEED_UPTIME PURPOSE=HS_VANGUARDS TIME_CREATED=2023-03-17T13:59:31.508480
13:59:32.723  D  CIRC: 5 EXTENDED $02B2152129686AE0BE05F3D1460591DD71C4BB15~0x67726579,$4F683B5B4F9A69F06AD9E83131B4678D4F1DDF78~thepinginthenorth,$8885EA6F74A694825B13B8A7080F6CF164DF74FB~Unnamed,$99D5C9A5A7238986D1EA1D7771CF81B4A192C4D0~arbitraryKenzie0 BUILD_FLAGS=IS_INTERNAL,NEED_UPTIME PURPOSE=HS_SERVICE_INTRO HS_STATE=HSSI_CONNECTING REND_QUERY=qgsg2fjxfibd462aaa3tzdbwtzf4uzx6gohp563oiyzbyh4fbcnev4ad TIME_CREATED=2023-03-17T13:59:31.308569
13:59:32.723  D  CIRC: 5 BUILT $02B2152129686AE0BE05F3D1460591DD71C4BB15~0x67726579,$4F683B5B4F9A69F06AD9E83131B4678D4F1DDF78~thepinginthenorth,$8885EA6F74A694825B13B8A7080F6CF164DF74FB~Unnamed,$99D5C9A5A7238986D1EA1D7771CF81B4A192C4D0~arbitraryKenzie0 BUILD_FLAGS=IS_INTERNAL,NEED_UPTIME PURPOSE=HS_SERVICE_INTRO HS_STATE=HSSI_CONNECTING REND_QUERY=qgsg2fjxfibd462aaa3tzdbwtzf4uzx6gohp563oiyzbyh4fbcnev4ad TIME_CREATED=2023-03-17T13:59:31.308569
13:59:32.723  D  CIRC: 5 BUILT $02B2152129686AE0BE05F3D1460591DD71C4BB15~0x67726579,$4F683B5B4F9A69F06AD9E83131B4678D4F1DDF78~thepinginthenorth,$8885EA6F74A694825B13B8A7080F6CF164DF74FB~Unnamed,$99D5C9A5A7238986D1EA1D7771CF81B4A192C4D0~arbitraryKenzie0 BUILD_FLAGS=IS_INTERNAL,NEED_UPTIME PURPOSE=HS_VANGUARDS TIME_CREATED=2023-03-17T13:59:31.308569
13:59:34.564  D  HS_DESC: UPLOAD qgsg2fjxfibd462aaa3tzdbwtzf4uzx6gohp563oiyzbyh4fbcnev4ad UNKNOWN $0E4F190C4A6F7DDF7F8C9A7842B85B4BBCF5F44B~ForPrivacyNET 2qWj37YKHL3ch0U9oCKV0twGIVvGH82qUKlEnMjXq7s HSDIR_INDEX=063CAA65D1EF2D73769CBDE4FE3023B9CD721149CB43C0578730497793E74224
13:59:34.564  D  HS_DESC: UPLOAD qgsg2fjxfibd462aaa3tzdbwtzf4uzx6gohp563oiyzbyh4fbcnev4ad UNKNOWN $828AA0934B50FAE6FDD5D63932420AE98F798A4D~rbnettor1 2qWj37YKHL3ch0U9oCKV0twGIVvGH82qUKlEnMjXq7s HSDIR_INDEX=0672FDA32371D6FD80A4FF3C8BCC5A4A44C1FF6C69C086C45E9079A65AB6EAC7
13:59:34.564  D  HS_DESC: UPLOAD qgsg2fjxfibd462aaa3tzdbwtzf4uzx6gohp563oiyzbyh4fbcnev4ad UNKNOWN $A50BC58C32D37168C040B50BE977F7F74DD81480~TheOneRelay 2qWj37YKHL3ch0U9oCKV0twGIVvGH82qUKlEnMjXq7s HSDIR_INDEX=067EFD852A894E3CF03415146B9F028E415F3F8C01A2715DF4041BAD622623F7
13:59:34.564  D  HS_DESC: UPLOAD qgsg2fjxfibd462aaa3tzdbwtzf4uzx6gohp563oiyzbyh4fbcnev4ad UNKNOWN $82423ACAA67A358A44771F637FF93701C729FA25~ididnteditheconfig 2qWj37YKHL3ch0U9oCKV0twGIVvGH82qUKlEnMjXq7s HSDIR_INDEX=067F8268722C5FEB1CFDC7DA89F9182CC365B97E43C8B3B72C89EFB36572FEBE
13:59:34.564  D  HS_DESC: UPLOAD qgsg2fjxfibd462aaa3tzdbwtzf4uzx6gohp563oiyzbyh4fbcnev4ad UNKNOWN $741288B92BC9FF14CB48102FC65927369E4B0B1C~MinchiNodes 2qWj37YKHL3ch0U9oCKV0twGIVvGH82qUKlEnMjXq7s HSDIR_INDEX=29D17CD67DFDE6576960DCF2F5756867CB284BA59F6F5C538EFA7A6609843F13
13:59:34.564  D  HS_DESC: UPLOAD qgsg2fjxfibd462aaa3tzdbwtzf4uzx6gohp563oiyzbyh4fbcnev4ad UNKNOWN $0F3874C18BE50B83939D09AF2F6C362D1FC6C8CD~Polyphemus5 2qWj37YKHL3ch0U9oCKV0twGIVvGH82qUKlEnMjXq7s HSDIR_INDEX=29D21217D75381CDD06579204CB360748BAB93D3E585FA625BB040F53A8662E2
13:59:34.565  D  HS_DESC: UPLOAD qgsg2fjxfibd462aaa3tzdbwtzf4uzx6gohp563oiyzbyh4fbcnev4ad UNKNOWN $81B8222C107CC0AAB575B83BF98D2492DB1F609D~ClankC 2qWj37YKHL3ch0U9oCKV0twGIVvGH82qUKlEnMjXq7s HSDIR_INDEX=29D6DAD6224951AE38B4B0BA5C1EBDE409B21609F7AFDC35F081C48EB4F1E17C
13:59:34.565  D  HS_DESC: UPLOAD qgsg2fjxfibd462aaa3tzdbwtzf4uzx6gohp563oiyzbyh4fbcnev4ad UNKNOWN $21A9BA4BB24D7A7E3DB96DF7D81C13016F3C3F47~tendies4 2qWj37YKHL3ch0U9oCKV0twGIVvGH82qUKlEnMjXq7s HSDIR_INDEX=29D7159B04F21A980431417226BC34C8FA31A700DF38DDE8381B70AD146DF2D1
13:59:34.565  D  HS_DESC: UPLOAD qgsg2fjxfibd462aaa3tzdbwtzf4uzx6gohp563oiyzbyh4fbcnev4ad UNKNOWN $225B6EAC75A89D30BAC2FCC7182D0187C1C4D762~HermanRimm fUlWBCtARnP0bDVp8sDIAzrewrK7KdR+A9qpof2KzXE HSDIR_INDEX=9786F40A73280FF28468C325A9224F3861B3E13C24E8DA85D26D1F100C5324B2
13:59:34.565  D  HS_DESC: UPLOAD qgsg2fjxfibd462aaa3tzdbwtzf4uzx6gohp563oiyzbyh4fbcnev4ad UNKNOWN $F72AF9F63CD443B2CC7FEB8552EC588EF1992F0A~sklep fUlWBCtARnP0bDVp8sDIAzrewrK7KdR+A9qpof2KzXE HSDIR_INDEX=978A6EF40D7E80B4B22ED85CCF1E364AADA016257B9A977DC44144CEF319AEBE
13:59:34.565  D  HS_DESC: UPLOAD qgsg2fjxfibd462aaa3tzdbwtzf4uzx6gohp563oiyzbyh4fbcnev4ad UNKNOWN $A761D2D04D301EE6BE3B0001A3C601FD857E6CA0~panda fUlWBCtARnP0bDVp8sDIAzrewrK7KdR+A9qpof2KzXE HSDIR_INDEX=978FB2BD8FB67C981858B32CEE649A7D10EC3EF18F1D90BD9EF48773ECEB34D4
13:59:34.565  D  HS_DESC: UPLOAD qgsg2fjxfibd462aaa3tzdbwtzf4uzx6gohp563oiyzbyh4fbcnev4ad UNKNOWN $2F09BD6D9A2D5A7D6D26C17651CE8ECF0B7BA257~TOR2DFN01b fUlWBCtARnP0bDVp8sDIAzrewrK7KdR+A9qpof2KzXE HSDIR_INDEX=97AC0CDF6F6D740D2604B7FAF9EEB855A87399D70380ABCFB5EC4F8F6CABF972
13:59:34.565  D  HS_DESC: UPLOAD qgsg2fjxfibd462aaa3tzdbwtzf4uzx6gohp563oiyzbyh4fbcnev4ad UNKNOWN $D496DD0675ED5396A184454E9594C76734C23323~CantThinkOfOne fUlWBCtARnP0bDVp8sDIAzrewrK7KdR+A9qpof2KzXE HSDIR_INDEX=16E940B334845D6A2C4FA73716957ABC6AA8989C5B9FDAD183897E09E6480204
13:59:34.565  D  HS_DESC: UPLOAD qgsg2fjxfibd462aaa3tzdbwtzf4uzx6gohp563oiyzbyh4fbcnev4ad UNKNOWN $1F772DD93DA20A6745E334BAFFC7B9765876BB11~ballers1 fUlWBCtARnP0bDVp8sDIAzrewrK7KdR+A9qpof2KzXE HSDIR_INDEX=170128AE758E53101A5B414C5C02F0C2F10891A75F6FFAA3AE4428916D1D3D1F
13:59:34.566  D  HS_DESC: UPLOAD qgsg2fjxfibd462aaa3tzdbwtzf4uzx6gohp563oiyzbyh4fbcnev4ad UNKNOWN $FC5B4F90D1A9BB6B5885CA6B4F4EC47D1783952D~GhostNetKennel fUlWBCtARnP0bDVp8sDIAzrewrK7KdR+A9qpof2KzXE HSDIR_INDEX=170505D3763CF36AFC7E2EC4B541A2D19746B3C9DC818F281F967F205AC7168D
13:59:34.566  D  HS_DESC: UPLOAD qgsg2fjxfibd462aaa3tzdbwtzf4uzx6gohp563oiyzbyh4fbcnev4ad UNKNOWN $F00B80BB580E1F78BC1E37BAF5B6FDED67E4DC3A~cloudyDC fUlWBCtARnP0bDVp8sDIAzrewrK7KdR+A9qpof2KzXE HSDIR_INDEX=170DD74DE3A6B52026C22B0DACB07A5EE73F25C0C44AD7EBB3191FA88F4132BD
13:59:34.566  D  CIRC: 15 LAUNCHED BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:33.598430
13:59:34.566  D  CIRC: 16 LAUNCHED BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:33.662834
13:59:34.566  D  CIRC: 17 LAUNCHED BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:33.726250
13:59:34.566  D  CIRC: 18 LAUNCHED BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:33.789730
13:59:34.566  D  CIRC: 19 LAUNCHED BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:33.853800
13:59:34.566  D  CIRC: 20 LAUNCHED BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:33.917160
13:59:34.566  D  CIRC: 21 LAUNCHED BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:33.981445
13:59:34.566  D  CIRC: 22 LAUNCHED BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:34.045113
13:59:34.567  D  CIRC: 23 LAUNCHED BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:34.108888
13:59:34.567  D  CIRC: 24 LAUNCHED BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:34.172931
13:59:34.567  D  CIRC: 25 LAUNCHED BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:34.245966
13:59:34.567  D  CIRC: 26 LAUNCHED BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:34.309270
13:59:34.567  D  CIRC: 27 LAUNCHED BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:34.372195
13:59:34.567  D  CIRC: 28 LAUNCHED BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:34.435199
13:59:34.567  D  CIRC: 29 LAUNCHED BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:34.498843
13:59:34.571  I  New state from: f-Zipped t-Starting w-Started 
13:59:34.571  D  New state: Starting
13:59:34.800  D  CIRC: 15 EXTENDED $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2 BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:33.598430
13:59:34.800  D  CIRC: 16 EXTENDED $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2 BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:33.662834
13:59:34.800  D  CIRC: 24 EXTENDED $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2 BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:34.172931
13:59:34.800  D  CIRC: 27 EXTENDED $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2 BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:34.372195
13:59:34.800  D  CIRC: 28 EXTENDED $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2 BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:34.435199
13:59:34.800  D  CIRC: 29 EXTENDED $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2 BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:34.498843
13:59:34.800  D  CIRC: 18 EXTENDED $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2 BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:33.789730
13:59:34.853  D  CIRC: 26 EXTENDED $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2 BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:34.309270
13:59:34.853  D  CIRC: 23 EXTENDED $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2 BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:34.108888
13:59:34.853  D  CIRC: 19 EXTENDED $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2 BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:33.853800
13:59:34.853  D  CIRC: 17 EXTENDED $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2 BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:33.726250
13:59:34.853  D  CIRC: 22 EXTENDED $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2 BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:34.045113
13:59:34.853  D  CIRC: 20 EXTENDED $02B2152129686AE0BE05F3D1460591DD71C4BB15~0x67726579 BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:33.917160
13:59:34.853  D  CIRC: 25 EXTENDED $02B2152129686AE0BE05F3D1460591DD71C4BB15~0x67726579 BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:34.245966
13:59:34.853  D  CIRC: 21 EXTENDED $02B2152129686AE0BE05F3D1460591DD71C4BB15~0x67726579 BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:33.981445
13:59:34.854  D  CIRC: 14 EXTENDED $02B2152129686AE0BE05F3D1460591DD71C4BB15~0x67726579,$4F683B5B4F9A69F06AD9E83131B4678D4F1DDF78~thepinginthenorth,$82A4BD0933B59E5A6F94BEA01A8EF55ABC031725~operator,$0E4F190C4A6F7DDF7F8C9A7842B85B4BBCF5F44B~ForPrivacyNET BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY,NEED_UPTIME PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:31.881767
13:59:34.854  D  CIRC: 14 BUILT $02B2152129686AE0BE05F3D1460591DD71C4BB15~0x67726579,$4F683B5B4F9A69F06AD9E83131B4678D4F1DDF78~thepinginthenorth,$82A4BD0933B59E5A6F94BEA01A8EF55ABC031725~operator,$0E4F190C4A6F7DDF7F8C9A7842B85B4BBCF5F44B~ForPrivacyNET BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY,NEED_UPTIME PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:31.881767
13:59:34.882  D  CIRC: 15 EXTENDED $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2,$82A4BD0933B59E5A6F94BEA01A8EF55ABC031725~operator BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:33.598430
13:59:34.882  D  CIRC: 27 EXTENDED $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2,$82A4BD0933B59E5A6F94BEA01A8EF55ABC031725~operator BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:34.372195
13:59:34.882  D  CIRC: 28 EXTENDED $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2,$4F683B5B4F9A69F06AD9E83131B4678D4F1DDF78~thepinginthenorth BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:34.435199
13:59:34.882  D  CIRC: 16 EXTENDED $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2,$A511F574066CDC228C79ED366007644995136BD8~iWantedToCancel BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:33.662834
13:59:34.882  D  CIRC: 18 EXTENDED $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2,$A511F574066CDC228C79ED366007644995136BD8~iWantedToCancel BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:33.789730
13:59:34.883  D  CIRC: 29 EXTENDED $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2,$C237392DC121DF50C64C707C3FB95055E68B00CA~pacoCalienteRelay2 BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:34.498843
13:59:34.883  D  CIRC: 24 EXTENDED $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2,$A511F574066CDC228C79ED366007644995136BD8~iWantedToCancel BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:34.172931
13:59:34.936  D  CIRC: 26 EXTENDED $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2,$82A4BD0933B59E5A6F94BEA01A8EF55ABC031725~operator BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:34.309270
13:59:34.936  D  CIRC: 23 EXTENDED $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2,$82A4BD0933B59E5A6F94BEA01A8EF55ABC031725~operator BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:34.108888
13:59:34.936  D  CIRC: 17 EXTENDED $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2,$82A4BD0933B59E5A6F94BEA01A8EF55ABC031725~operator BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:33.726250
13:59:34.993  D  CIRC: 22 EXTENDED $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2,$4F683B5B4F9A69F06AD9E83131B4678D4F1DDF78~thepinginthenorth BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:34.045113
13:59:34.993  D  CIRC: 19 EXTENDED $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2,$4F683B5B4F9A69F06AD9E83131B4678D4F1DDF78~thepinginthenorth BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:33.853800
13:59:34.993  D  CIRC: 20 EXTENDED $02B2152129686AE0BE05F3D1460591DD71C4BB15~0x67726579,$82A4BD0933B59E5A6F94BEA01A8EF55ABC031725~operator BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:33.917160
13:59:34.994  D  CIRC: 25 EXTENDED $02B2152129686AE0BE05F3D1460591DD71C4BB15~0x67726579,$A511F574066CDC228C79ED366007644995136BD8~iWantedToCancel BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:34.245966
13:59:34.994  D  CIRC: 21 EXTENDED $02B2152129686AE0BE05F3D1460591DD71C4BB15~0x67726579,$C237392DC121DF50C64C707C3FB95055E68B00CA~pacoCalienteRelay2 BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:33.981445
13:59:35.027  D  CIRC: 15 EXTENDED $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2,$82A4BD0933B59E5A6F94BEA01A8EF55ABC031725~operator,$1B7F47827E2FFAFE05B92B1C5A92B8F9A4AF67DD~Vulnerar BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:33.598430
13:59:35.028  D  CIRC: 18 EXTENDED $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2,$A511F574066CDC228C79ED366007644995136BD8~iWantedToCancel,$97E4BAB582717350E6FF6160899FA9B665C834E3~LSTorRelay01 BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:33.789730
13:59:35.028  D  CIRC: 16 EXTENDED $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2,$A511F574066CDC228C79ED366007644995136BD8~iWantedToCancel,$FAD8CB4C7CFBF38E37E26164C47B13624417B3D7~BadGateway BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:33.662834
13:59:35.028  D  CIRC: 24 EXTENDED $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2,$A511F574066CDC228C79ED366007644995136BD8~iWantedToCancel,$9371FF8BBDD6E2BF0ED88251B490E0B9783C1BA0~Achillobator BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:34.172931
13:59:35.047  D  CIRC: 17 EXTENDED $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2,$82A4BD0933B59E5A6F94BEA01A8EF55ABC031725~operator,$5C426791CB47BDDDD97A71EB2EFC4CCC84DEB299~ReacTOR BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:33.726250
13:59:35.047  D  CIRC: 29 EXTENDED $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2,$C237392DC121DF50C64C707C3FB95055E68B00CA~pacoCalienteRelay2,$DF6F7908E7FDEEE5DE6DD9C92F7C9DFB20B0CE2A~TorchWood BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:34.498843
13:59:35.048  D  CIRC: 23 EXTENDED $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2,$82A4BD0933B59E5A6F94BEA01A8EF55ABC031725~operator,$A751D8BEE222564B7BA657F74B6658836FF1AEEA~ForPrivacyNET BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:34.108888
13:59:35.048  D  CIRC: 26 EXTENDED $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2,$82A4BD0933B59E5A6F94BEA01A8EF55ABC031725~operator,$A15FD725CD3D816B855000E7293BF97409C31410~prsv BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:34.309270
13:59:35.048  D  CIRC: 27 EXTENDED $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2,$82A4BD0933B59E5A6F94BEA01A8EF55ABC031725~operator,$11083713EC88FF09269EC37F5AD9DDAACA2F247C~cricket BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:34.372195
13:59:35.048  D  HS_DESC: UPLOADED qgsg2fjxfibd462aaa3tzdbwtzf4uzx6gohp563oiyzbyh4fbcnev4ad UNKNOWN $0E4F190C4A6F7DDF7F8C9A7842B85B4BBCF5F44B~ForPrivacyNET
13:59:35.049  D  CIRC: 28 EXTENDED $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2,$4F683B5B4F9A69F06AD9E83131B4678D4F1DDF78~thepinginthenorth,$304EF9E012BE51190C6DDD5CFBAD8E3F3E875A43~TorRelay2c BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:34.435199
13:59:35.050  D  CIRC: 22 EXTENDED $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2,$4F683B5B4F9A69F06AD9E83131B4678D4F1DDF78~thepinginthenorth,$874D84382C892F3F61CC9E106BF08843DE0B865A~Spigen BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:34.045113
13:59:35.052  I  New state from: f-Zipped t-Started w-Started 
13:59:35.054  D  New state: Sharing
13:59:35.196  W  requestLayout() improperly called by androidx.compose.ui.platform.ViewLayerContainer{1597eae6 V.E..... ......ID 0,0-0,0} during layout: running second layout pass
13:59:35.202  D  CIRC: 19 EXTENDED $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2,$4F683B5B4F9A69F06AD9E83131B4678D4F1DDF78~thepinginthenorth,$9C3EE508E58DD9A4240BB0CD8BF1AA87807BED6E~STOR005 BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:33.853800
13:59:35.202  D  CIRC: 15 EXTENDED $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2,$82A4BD0933B59E5A6F94BEA01A8EF55ABC031725~operator,$1B7F47827E2FFAFE05B92B1C5A92B8F9A4AF67DD~Vulnerar,$828AA0934B50FAE6FDD5D63932420AE98F798A4D~rbnettor1 BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:33.598430
13:59:35.203  D  CIRC: 15 BUILT $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2,$82A4BD0933B59E5A6F94BEA01A8EF55ABC031725~operator,$1B7F47827E2FFAFE05B92B1C5A92B8F9A4AF67DD~Vulnerar,$828AA0934B50FAE6FDD5D63932420AE98F798A4D~rbnettor1 BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:33.598430
13:59:35.584  D  CIRC: 16 EXTENDED $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2,$A511F574066CDC228C79ED366007644995136BD8~iWantedToCancel,$FAD8CB4C7CFBF38E37E26164C47B13624417B3D7~BadGateway,$A50BC58C32D37168C040B50BE977F7F74DD81480~TheOneRelay BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:33.662834
13:59:35.584  D  CIRC: 16 BUILT $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2,$A511F574066CDC228C79ED366007644995136BD8~iWantedToCancel,$FAD8CB4C7CFBF38E37E26164C47B13624417B3D7~BadGateway,$A50BC58C32D37168C040B50BE977F7F74DD81480~TheOneRelay BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:33.662834
13:59:35.585  D  CIRC: 20 EXTENDED $02B2152129686AE0BE05F3D1460591DD71C4BB15~0x67726579,$82A4BD0933B59E5A6F94BEA01A8EF55ABC031725~operator,$5D575ADD108DDBBF17E78AD8CB267D6C1F338206~Hydra93a BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:33.917160
13:59:35.585  D  CIRC: 25 EXTENDED $02B2152129686AE0BE05F3D1460591DD71C4BB15~0x67726579,$A511F574066CDC228C79ED366007644995136BD8~iWantedToCancel,$0B06DD6A65C48EA79FC43031CBD93650B08A0661~bauruine BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:34.245966
13:59:36.023  D  CIRC: 17 EXTENDED $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2,$82A4BD0933B59E5A6F94BEA01A8EF55ABC031725~operator,$5C426791CB47BDDDD97A71EB2EFC4CCC84DEB299~ReacTOR,$82423ACAA67A358A44771F637FF93701C729FA25~ididnteditheconfig BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:33.726250
13:59:36.023  D  CIRC: 17 BUILT $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2,$82A4BD0933B59E5A6F94BEA01A8EF55ABC031725~operator,$5C426791CB47BDDDD97A71EB2EFC4CCC84DEB299~ReacTOR,$82423ACAA67A358A44771F637FF93701C729FA25~ididnteditheconfig BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:33.726250
13:59:36.023  D  CIRC: 22 EXTENDED $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2,$4F683B5B4F9A69F06AD9E83131B4678D4F1DDF78~thepinginthenorth,$874D84382C892F3F61CC9E106BF08843DE0B865A~Spigen,$225B6EAC75A89D30BAC2FCC7182D0187C1C4D762~HermanRimm BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:34.045113
13:59:36.023  D  CIRC: 22 BUILT $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2,$4F683B5B4F9A69F06AD9E83131B4678D4F1DDF78~thepinginthenorth,$874D84382C892F3F61CC9E106BF08843DE0B865A~Spigen,$225B6EAC75A89D30BAC2FCC7182D0187C1C4D762~HermanRimm BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:34.045113
13:59:36.024  D  CIRC: 24 EXTENDED $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2,$A511F574066CDC228C79ED366007644995136BD8~iWantedToCancel,$9371FF8BBDD6E2BF0ED88251B490E0B9783C1BA0~Achillobator,$A761D2D04D301EE6BE3B0001A3C601FD857E6CA0~panda BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:34.172931
13:59:36.024  D  CIRC: 24 BUILT $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2,$A511F574066CDC228C79ED366007644995136BD8~iWantedToCancel,$9371FF8BBDD6E2BF0ED88251B490E0B9783C1BA0~Achillobator,$A761D2D04D301EE6BE3B0001A3C601FD857E6CA0~panda BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:34.172931
13:59:36.024  D  CIRC: 23 EXTENDED $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2,$82A4BD0933B59E5A6F94BEA01A8EF55ABC031725~operator,$A751D8BEE222564B7BA657F74B6658836FF1AEEA~ForPrivacyNET,$F72AF9F63CD443B2CC7FEB8552EC588EF1992F0A~sklep BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:34.108888
13:59:36.024  D  CIRC: 23 BUILT $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2,$82A4BD0933B59E5A6F94BEA01A8EF55ABC031725~operator,$A751D8BEE222564B7BA657F74B6658836FF1AEEA~ForPrivacyNET,$F72AF9F63CD443B2CC7FEB8552EC588EF1992F0A~sklep BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:34.108888
13:59:36.197  D  CIRC: 18 EXTENDED $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2,$A511F574066CDC228C79ED366007644995136BD8~iWantedToCancel,$97E4BAB582717350E6FF6160899FA9B665C834E3~LSTorRelay01,$741288B92BC9FF14CB48102FC65927369E4B0B1C~MinchiNodes BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:33.789730
13:59:36.198  D  CIRC: 18 BUILT $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2,$A511F574066CDC228C79ED366007644995136BD8~iWantedToCancel,$97E4BAB582717350E6FF6160899FA9B665C834E3~LSTorRelay01,$741288B92BC9FF14CB48102FC65927369E4B0B1C~MinchiNodes BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:33.789730
13:59:36.198  D  CIRC: 27 EXTENDED $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2,$82A4BD0933B59E5A6F94BEA01A8EF55ABC031725~operator,$11083713EC88FF09269EC37F5AD9DDAACA2F247C~cricket,$1F772DD93DA20A6745E334BAFFC7B9765876BB11~ballers1 BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:34.372195
13:59:36.198  D  CIRC: 27 BUILT $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2,$82A4BD0933B59E5A6F94BEA01A8EF55ABC031725~operator,$11083713EC88FF09269EC37F5AD9DDAACA2F247C~cricket,$1F772DD93DA20A6745E334BAFFC7B9765876BB11~ballers1 BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:34.372195
13:59:36.198  D  CIRC: 19 EXTENDED $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2,$4F683B5B4F9A69F06AD9E83131B4678D4F1DDF78~thepinginthenorth,$9C3EE508E58DD9A4240BB0CD8BF1AA87807BED6E~STOR005,$0F3874C18BE50B83939D09AF2F6C362D1FC6C8CD~Polyphemus5 BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:33.853800
13:59:36.198  D  CIRC: 19 BUILT $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2,$4F683B5B4F9A69F06AD9E83131B4678D4F1DDF78~thepinginthenorth,$9C3EE508E58DD9A4240BB0CD8BF1AA87807BED6E~STOR005,$0F3874C18BE50B83939D09AF2F6C362D1FC6C8CD~Polyphemus5 BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:33.853800
13:59:36.198  D  CIRC: 26 EXTENDED $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2,$82A4BD0933B59E5A6F94BEA01A8EF55ABC031725~operator,$A15FD725CD3D816B855000E7293BF97409C31410~prsv,$D496DD0675ED5396A184454E9594C76734C23323~CantThinkOfOne BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:34.309270
13:59:36.198  D  CIRC: 26 BUILT $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2,$82A4BD0933B59E5A6F94BEA01A8EF55ABC031725~operator,$A15FD725CD3D816B855000E7293BF97409C31410~prsv,$D496DD0675ED5396A184454E9594C76734C23323~CantThinkOfOne BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:34.309270
13:59:36.198  D  CIRC: 28 EXTENDED $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2,$4F683B5B4F9A69F06AD9E83131B4678D4F1DDF78~thepinginthenorth,$304EF9E012BE51190C6DDD5CFBAD8E3F3E875A43~TorRelay2c,$FC5B4F90D1A9BB6B5885CA6B4F4EC47D1783952D~GhostNetKennel BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:34.435199
13:59:36.198  D  CIRC: 28 BUILT $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2,$4F683B5B4F9A69F06AD9E83131B4678D4F1DDF78~thepinginthenorth,$304EF9E012BE51190C6DDD5CFBAD8E3F3E875A43~TorRelay2c,$FC5B4F90D1A9BB6B5885CA6B4F4EC47D1783952D~GhostNetKennel BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:34.435199
13:59:36.199  D  CIRC: 29 EXTENDED $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2,$C237392DC121DF50C64C707C3FB95055E68B00CA~pacoCalienteRelay2,$DF6F7908E7FDEEE5DE6DD9C92F7C9DFB20B0CE2A~TorchWood,$F00B80BB580E1F78BC1E37BAF5B6FDED67E4DC3A~cloudyDC BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:34.498843
13:59:36.201  D  CIRC: 29 BUILT $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2,$C237392DC121DF50C64C707C3FB95055E68B00CA~pacoCalienteRelay2,$DF6F7908E7FDEEE5DE6DD9C92F7C9DFB20B0CE2A~TorchWood,$F00B80BB580E1F78BC1E37BAF5B6FDED67E4DC3A~cloudyDC BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:34.498843
13:59:36.201  D  CIRC: 21 EXTENDED $02B2152129686AE0BE05F3D1460591DD71C4BB15~0x67726579,$C237392DC121DF50C64C707C3FB95055E68B00CA~pacoCalienteRelay2,$6F95E6C518BD09EA49A31025A7B11FD854FA6A9E~sunandfun01 BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:33.981445
13:59:36.201  D  CIRC: 25 EXTENDED $02B2152129686AE0BE05F3D1460591DD71C4BB15~0x67726579,$A511F574066CDC228C79ED366007644995136BD8~iWantedToCancel,$0B06DD6A65C48EA79FC43031CBD93650B08A0661~bauruine,$2F09BD6D9A2D5A7D6D26C17651CE8ECF0B7BA257~TOR2DFN01b BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:34.245966
13:59:36.201  D  CIRC: 25 BUILT $02B2152129686AE0BE05F3D1460591DD71C4BB15~0x67726579,$A511F574066CDC228C79ED366007644995136BD8~iWantedToCancel,$0B06DD6A65C48EA79FC43031CBD93650B08A0661~bauruine,$2F09BD6D9A2D5A7D6D26C17651CE8ECF0B7BA257~TOR2DFN01b BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:34.245966
13:59:36.201  D  CIRC: 20 EXTENDED $02B2152129686AE0BE05F3D1460591DD71C4BB15~0x67726579,$82A4BD0933B59E5A6F94BEA01A8EF55ABC031725~operator,$5D575ADD108DDBBF17E78AD8CB267D6C1F338206~Hydra93a,$81B8222C107CC0AAB575B83BF98D2492DB1F609D~ClankC BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:33.917160
13:59:36.201  D  CIRC: 20 BUILT $02B2152129686AE0BE05F3D1460591DD71C4BB15~0x67726579,$82A4BD0933B59E5A6F94BEA01A8EF55ABC031725~operator,$5D575ADD108DDBBF17E78AD8CB267D6C1F338206~Hydra93a,$81B8222C107CC0AAB575B83BF98D2492DB1F609D~ClankC BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:33.917160
13:59:36.211  D  HS_DESC: UPLOADED qgsg2fjxfibd462aaa3tzdbwtzf4uzx6gohp563oiyzbyh4fbcnev4ad UNKNOWN $828AA0934B50FAE6FDD5D63932420AE98F798A4D~rbnettor1
13:59:36.211  D  HS_DESC: UPLOADED qgsg2fjxfibd462aaa3tzdbwtzf4uzx6gohp563oiyzbyh4fbcnev4ad UNKNOWN $A50BC58C32D37168C040B50BE977F7F74DD81480~TheOneRelay
13:59:36.477  D  HS_DESC: UPLOADED qgsg2fjxfibd462aaa3tzdbwtzf4uzx6gohp563oiyzbyh4fbcnev4ad UNKNOWN $2F09BD6D9A2D5A7D6D26C17651CE8ECF0B7BA257~TOR2DFN01b
13:59:36.529  D  CIRC: 21 EXTENDED $02B2152129686AE0BE05F3D1460591DD71C4BB15~0x67726579,$C237392DC121DF50C64C707C3FB95055E68B00CA~pacoCalienteRelay2,$6F95E6C518BD09EA49A31025A7B11FD854FA6A9E~sunandfun01,$21A9BA4BB24D7A7E3DB96DF7D81C13016F3C3F47~tendies4 BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:33.981445
13:59:36.529  D  CIRC: 21 BUILT $02B2152129686AE0BE05F3D1460591DD71C4BB15~0x67726579,$C237392DC121DF50C64C707C3FB95055E68B00CA~pacoCalienteRelay2,$6F95E6C518BD09EA49A31025A7B11FD854FA6A9E~sunandfun01,$21A9BA4BB24D7A7E3DB96DF7D81C13016F3C3F47~tendies4 BUILD_FLAGS=IS_INTERNAL,NEED_CAPACITY PURPOSE=HS_SERVICE_HSDIR HS_STATE=HSSI_CONNECTING TIME_CREATED=2023-03-17T13:59:33.981445
13:59:36.551  D  HS_DESC: UPLOADED qgsg2fjxfibd462aaa3tzdbwtzf4uzx6gohp563oiyzbyh4fbcnev4ad UNKNOWN $81B8222C107CC0AAB575B83BF98D2492DB1F609D~ClankC
13:59:36.699  D  HS_DESC: UPLOADED qgsg2fjxfibd462aaa3tzdbwtzf4uzx6gohp563oiyzbyh4fbcnev4ad UNKNOWN $82423ACAA67A358A44771F637FF93701C729FA25~ididnteditheconfig
13:59:36.700  D  HS_DESC: UPLOADED qgsg2fjxfibd462aaa3tzdbwtzf4uzx6gohp563oiyzbyh4fbcnev4ad UNKNOWN $F72AF9F63CD443B2CC7FEB8552EC588EF1992F0A~sklep
13:59:36.700  D  HS_DESC: UPLOADED qgsg2fjxfibd462aaa3tzdbwtzf4uzx6gohp563oiyzbyh4fbcnev4ad UNKNOWN $225B6EAC75A89D30BAC2FCC7182D0187C1C4D762~HermanRimm
13:59:36.727  D  HS_DESC: UPLOADED qgsg2fjxfibd462aaa3tzdbwtzf4uzx6gohp563oiyzbyh4fbcnev4ad UNKNOWN $A761D2D04D301EE6BE3B0001A3C601FD857E6CA0~panda
13:59:36.805  D  HS_DESC: UPLOADED qgsg2fjxfibd462aaa3tzdbwtzf4uzx6gohp563oiyzbyh4fbcnev4ad UNKNOWN $741288B92BC9FF14CB48102FC65927369E4B0B1C~MinchiNodes
13:59:36.846  D  HS_DESC: UPLOADED qgsg2fjxfibd462aaa3tzdbwtzf4uzx6gohp563oiyzbyh4fbcnev4ad UNKNOWN $1F772DD93DA20A6745E334BAFFC7B9765876BB11~ballers1
13:59:36.865  D  HS_DESC: UPLOADED qgsg2fjxfibd462aaa3tzdbwtzf4uzx6gohp563oiyzbyh4fbcnev4ad UNKNOWN $D496DD0675ED5396A184454E9594C76734C23323~CantThinkOfOne
13:59:36.897  D  HS_DESC: UPLOADED qgsg2fjxfibd462aaa3tzdbwtzf4uzx6gohp563oiyzbyh4fbcnev4ad UNKNOWN $0F3874C18BE50B83939D09AF2F6C362D1FC6C8CD~Polyphemus5
13:59:37.016  D  HS_DESC: UPLOADED qgsg2fjxfibd462aaa3tzdbwtzf4uzx6gohp563oiyzbyh4fbcnev4ad UNKNOWN $FC5B4F90D1A9BB6B5885CA6B4F4EC47D1783952D~GhostNetKennel
13:59:37.061  D  HS_DESC: UPLOADED qgsg2fjxfibd462aaa3tzdbwtzf4uzx6gohp563oiyzbyh4fbcnev4ad UNKNOWN $21A9BA4BB24D7A7E3DB96DF7D81C13016F3C3F47~tendies4
13:59:37.722  D  HS_DESC: UPLOADED qgsg2fjxfibd462aaa3tzdbwtzf4uzx6gohp563oiyzbyh4fbcnev4ad UNKNOWN $F00B80BB580E1F78BC1E37BAF5B6FDED67E4DC3A~cloudyDC
13:59:43.669  D  CIRC: 30 LAUNCHED BUILD_FLAGS=NEED_CAPACITY PURPOSE=GENERAL TIME_CREATED=2023-03-17T13:59:43.597330
13:59:43.714  D  CIRC: 30 EXTENDED $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2 BUILD_FLAGS=NEED_CAPACITY PURPOSE=GENERAL TIME_CREATED=2023-03-17T13:59:43.597330
13:59:43.768  D  CIRC: 30 EXTENDED $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2,$8B4ECB759D13BF6778ACAA52D666EB2A3E7D2EED~Spellunking BUILD_FLAGS=NEED_CAPACITY PURPOSE=GENERAL TIME_CREATED=2023-03-17T13:59:43.597330
13:59:43.876  D  CIRC: 30 EXTENDED $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2,$8B4ECB759D13BF6778ACAA52D666EB2A3E7D2EED~Spellunking,$BAB161E1E3319F005568E27029681ED537AF3406~relayon1155 BUILD_FLAGS=NEED_CAPACITY PURPOSE=GENERAL TIME_CREATED=2023-03-17T13:59:43.597330
13:59:43.877  D  CIRC: 30 BUILT $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2,$8B4ECB759D13BF6778ACAA52D666EB2A3E7D2EED~Spellunking,$BAB161E1E3319F005568E27029681ED537AF3406~relayon1155 BUILD_FLAGS=NEED_CAPACITY PURPOSE=GENERAL TIME_CREATED=2023-03-17T13:59:43.597330
13:59:54.810  D  CIRC: 31 LAUNCHED BUILD_FLAGS=NEED_CAPACITY PURPOSE=GENERAL TIME_CREATED=2023-03-17T13:59:54.733447
13:59:54.859  D  CIRC: 31 EXTENDED $02B2152129686AE0BE05F3D1460591DD71C4BB15~0x67726579 BUILD_FLAGS=NEED_CAPACITY PURPOSE=GENERAL TIME_CREATED=2023-03-17T13:59:54.733447
13:59:54.952  D  CIRC: 31 EXTENDED $02B2152129686AE0BE05F3D1460591DD71C4BB15~0x67726579,$509EAB4C5D10C9A9A24B4EA0CE402C047A2D64E6~toralf2 BUILD_FLAGS=NEED_CAPACITY PURPOSE=GENERAL TIME_CREATED=2023-03-17T13:59:54.733447
13:59:55.057  D  CIRC: 31 EXTENDED $02B2152129686AE0BE05F3D1460591DD71C4BB15~0x67726579,$509EAB4C5D10C9A9A24B4EA0CE402C047A2D64E6~toralf2,$BAB161E1E3319F005568E27029681ED537AF3406~relayon1155 BUILD_FLAGS=NEED_CAPACITY PURPOSE=GENERAL TIME_CREATED=2023-03-17T13:59:54.733447
13:59:55.057  D  CIRC: 31 BUILT $02B2152129686AE0BE05F3D1460591DD71C4BB15~0x67726579,$509EAB4C5D10C9A9A24B4EA0CE402C047A2D64E6~toralf2,$BAB161E1E3319F005568E27029681ED537AF3406~relayon1155 BUILD_FLAGS=NEED_CAPACITY PURPOSE=GENERAL TIME_CREATED=2023-03-17T13:59:54.733447
14:00:05.847  D  CIRC: 32 LAUNCHED BUILD_FLAGS=NEED_CAPACITY PURPOSE=GENERAL TIME_CREATED=2023-03-17T14:00:05.756882
14:00:05.899  D  CIRC: 32 EXTENDED $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2 BUILD_FLAGS=NEED_CAPACITY PURPOSE=GENERAL TIME_CREATED=2023-03-17T14:00:05.756882
14:00:06.020  D  CIRC: 32 EXTENDED $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2,$CD689044D0E65F52D96A1ACB9B6D0F24C032E1AE~mars BUILD_FLAGS=NEED_CAPACITY PURPOSE=GENERAL TIME_CREATED=2023-03-17T14:00:05.756882
14:00:06.167  D  CIRC: 32 EXTENDED $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2,$CD689044D0E65F52D96A1ACB9B6D0F24C032E1AE~mars,$8EC720BE33B7F6D59A3495B400BF36DEA6B97D21~ingwer BUILD_FLAGS=NEED_CAPACITY PURPOSE=GENERAL TIME_CREATED=2023-03-17T14:00:05.756882
14:00:06.167  D  CIRC: 32 BUILT $71A1DA0D554EC4107DD6ED7303D38DC9A9F8B241~chaoscode2,$CD689044D0E65F52D96A1ACB9B6D0F24C032E1AE~mars,$8EC720BE33B7F6D59A3495B400BF36DEA6B97D21~ingwer BUILD_FLAGS=NEED_CAPACITY PURPOSE=GENERAL TIME_CREATED=2023-03-17T14:00:05.756882
14:00:09.154  I  Stopping sharing...
14:00:09.154  I  Stopping...
14:00:09.167  I  New state from: f-Zipped t-Stopping w-Started stop!
14:00:09.169  D  New state: Stopping
14:00:09.173  D  onDestroy
14:00:09.180  I  Stopping... (isFinishingDownloading: false)
14:00:09.196  I  Releasing lock
14:00:09.260  I  System.exit called, status: 0
14:00:09.260  I  VM exiting with result code 0, cleanup skipped.
14:00:09.286  D  TorService: Stopped
14:00:09.288  I  Stopped
14:00:09.290  I  New state from: f-Zipped t-Stopped w-Started stop!
---------------------------- PROCESS ENDED (12487) for package org.onionshare.android.nightly ----------------------------
14:00:09.290  E  Unhandled state: ↑
14:00:11.236  I  Application stopping: io.ktor.server.application.Application@169d5c69
14:00:11.238  I  Application stopped: io.ktor.server.application.Application@169d5c69
14:00:11.238  I  Stopped
14:00:11.243  I  New state from: f-Zipped t-Stopped w-Stopped stop!
14:00:11.243  E  Unhandled state: ↑
14:00:11.247  I  New state from: f-Added t-Stopped w-Stopped stop!
14:00:11.247  D  New state: FilesAdded
14:00:35.053  W  No such thread id for suspend: 20
14:00:35.055  W  No such thread id for suspend: 20
14:00:39.863  I  New state from: f-Added t-Stopped w-Stopped 
14:00:39.866  I  Starting...
14:00:39.873  I  New state from: f-Added t-Starting w-Stopped 
14:00:39.873  E  Unhandled state: ↑
14:00:39.878  I  New state from: f-Zipping t-Starting w-Stopped 
14:00:39.879  D  New state: Starting
---------------------------- PROCESS STARTED (12930) for package org.onionshare.android.nightly ----------------------------
14:00:40.157  W  libtor.so: unused DT entry: type 0x6ffffef5 arg 0x6b088
14:00:40.157  W  libtor.so: unused DT entry: type 0x6ffffffe arg 0x98504
14:00:40.157  W  libtor.so: unused DT entry: type 0x6fffffff arg 0x3
14:00:40.162  E  "libtor.so": ignoring 2-entry DT_PREINIT_ARRAY in shared library!
14:00:40.191  I  Acquired lock
14:00:40.270  D  TorService: Starting...
14:00:40.351  I  New state from: f-Zipping t-Starting w-Stopped 
14:00:40.351  D  New state: Starting
14:00:40.590  D  onStartCommand Intent { cmp=org.onionshare.android.nightly/org.onionshare.android.tor.ShareService }
14:00:40.622  D  TorService: Starting...
14:00:40.651  I  New state from: f-Zipping t-Starting w-Stopped 
14:00:40.651  D  New state: Starting
14:00:40.654  I  New state from: f-Zipped t-Starting w-Stopped 
14:00:40.656  I  Autoreload is disabled because the development mode is off.
14:00:40.663  I  Application started in 0.008 seconds.
14:00:40.663  I  Application started: io.ktor.server.application.Application@26fed4d0
14:00:44.881  E  FATAL EXCEPTION: DefaultDispatcher-worker-7
                 Process: org.onionshare.android.nightly, PID: 11705
                 java.net.BindException: bind failed: EADDRINUSE (Address already in use)
                 	at libcore.io.IoBridge.bind(IoBridge.java:99)
                 	at java.net.PlainSocketImpl.bind(PlainSocketImpl.java:132)
                 	at java.net.ServerSocket.bind(ServerSocket.java:335)
                 	at io.netty.channel.socket.nio.NioServerSocketChannel.doBind(NioServerSocketChannel.java:143)
                 	at io.netty.channel.AbstractChannel$AbstractUnsafe.bind(AbstractChannel.java:562)
                 	at io.netty.channel.DefaultChannelPipeline$HeadContext.bind(DefaultChannelPipeline.java:1334)
                 	at io.netty.channel.AbstractChannelHandlerContext.invokeBind(AbstractChannelHandlerContext.java:600)
                 	at io.netty.channel.AbstractChannelHandlerContext.bind(AbstractChannelHandlerContext.java:579)
                 	at io.netty.channel.DefaultChannelPipeline.bind(DefaultChannelPipeline.java:973)
                 	at io.netty.channel.AbstractChannel.bind(AbstractChannel.java:260)
                 	at io.netty.bootstrap.AbstractBootstrap$2.run(AbstractBootstrap.java:356)
                 	at io.netty.util.concurrent.AbstractEventExecutor.runTask(AbstractEventExecutor.java:174)
                 	at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:167)
                 	at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:470)
                 	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:569)
                 	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997)
                 	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
                 	at io.ktor.server.netty.EventLoopGroupProxy$Companion.create$lambda$1$lambda$0(NettyApplicationEngine.kt:291)
                 	at io.ktor.server.netty.EventLoopGroupProxy$Companion.$r8$lambda$XgnKz7L6tCWRUaIAa7SVURwtFE4(NettyApplicationEngine.kt)
                 	at io.ktor.server.netty.EventLoopGroupProxy$Companion$$ExternalSyntheticLambda1.run(D8$$SyntheticClass)
                 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
                 	at java.lang.Thread.run(Thread.java:818)
                 	Suppressed: kotlinx.coroutines.DiagnosticCoroutineContextException: [StandaloneCoroutine{Cancelling}@29d936c4, Dispatchers.Default]
                 Caused by: android.system.ErrnoException: bind failed: EADDRINUSE (Address already in use)
                 	at libcore.io.Posix.bind(Native Method)
                 	at libcore.io.ForwardingOs.bind(ForwardingOs.java:56)
                 	at libcore.io.IoBridge.bind(IoBridge.java:97)
                 	... 21 more
```

A couple of things I noticed in the log:
* "Unhandled state" is logged a few times - not sure if this is indicative of a problem?
* The web server moves to the Stopped state before being restarted, and yet we get EADDRINUSE when trying to restart it

I couldn't easily test whether this also happens on the main branch because of the issue with java.util.function.Function being unavailable.

Reply:
Here's the relevant part of the log from the API 33 device where the crash doesn't happen:

```
2023-03-17 14:54:17.929 30434-30813 o.o.a.ShareManager      org.onionshare.android.nightly       I  Stopping sharing...
2023-03-17 14:54:17.929 30434-30813 o.o.a.t.TorManager      org.onionshare.android.nightly       I  Stopping...
2023-03-17 14:54:17.931 30434-30592 o.o.a.ShareManager      org.onionshare.android.nightly       I  New state from: f-Zipped t-Started w-Started stop!
2023-03-17 14:54:17.932 30816-30816 o.o.a.t.ShareService    org.onionshare.android.nightly       D  onDestroy
2023-03-17 14:54:17.932 30434-30813 o.o.a.s.We...verManager org.onionshare.android.nightly       I  Stopping... (isFinishingDownloading: false)
2023-03-17 14:54:17.933 30816-30816 TorService              org.onionshare.android.nightly       I  Releasing lock
2023-03-17 14:54:17.937 30434-30592 o.o.a.ShareManager      org.onionshare.android.nightly       I  New state from: f-Zipped t-Stopping w-Started stop!
2023-03-17 14:54:17.938 30434-30592 o.o.a.ShareManager      org.onionshare.android.nightly       D  New state: Stopping
2023-03-17 14:54:17.949 30816-30816 oid.nightly:tor         org.onionshare.android.nightly       I  System.exit called, status: 0
---------------------------- PROCESS ENDED (30816) for package org.onionshare.android.nightly ----------------------------
2023-03-17 14:54:17.949 30816-30816 AndroidRuntime          org.onionshare.android.nightly       I  VM exiting with result code 0, cleanup skipped.
2023-03-17 14:54:18.447 30434-30434 o.o.a.t.TorManager      org.onionshare.android.nightly       D  TorService: Stopped
2023-03-17 14:54:18.448 30434-30434 o.o.a.t.TorManager      org.onionshare.android.nightly       I  Stopped
2023-03-17 14:54:18.449 30434-30593 o.o.a.ShareManager      org.onionshare.android.nightly       I  New state from: f-Zipped t-Stopped w-Started stop!
2023-03-17 14:54:18.449 30434-30593 o.o.a.ShareManager      org.onionshare.android.nightly       E  Unhandled state: ↑
2023-03-17 14:54:19.972 30434-30813 k.application           org.onionshare.android.nightly       I  Application stopping: io.ktor.server.application.Application@cf1ac63
2023-03-17 14:54:19.973 30434-30813 k.application           org.onionshare.android.nightly       I  Application stopped: io.ktor.server.application.Application@cf1ac63
2023-03-17 14:54:19.974 30434-30813 o.o.a.s.We...verManager org.onionshare.android.nightly       I  Stopped
2023-03-17 14:54:19.976 30434-30593 o.o.a.ShareManager      org.onionshare.android.nightly       I  New state from: f-Zipped t-Stopped w-Stopped stop!
2023-03-17 14:54:19.976 30434-30593 o.o.a.ShareManager      org.onionshare.android.nightly       E  Unhandled state: ↑
2023-03-17 14:54:19.977 30434-30593 o.o.a.ShareManager      org.onionshare.android.nightly       I  New state from: f-Added t-Stopped w-Stopped stop!
2023-03-17 14:54:19.978 30434-30593 o.o.a.ShareManager      org.onionshare.android.nightly       D  New state: FilesAdded
2023-03-17 14:54:24.263 30434-30593 o.o.a.t.TorManager      org.onionshare.android.nightly       I  Starting...
2023-03-17 14:54:24.263 30434-30592 o.o.a.ShareManager      org.onionshare.android.nightly       I  New state from: f-Added t-Stopped w-Stopped 
2023-03-17 14:54:24.265 30434-30592 o.o.a.ShareManager      org.onionshare.android.nightly       I  New state from: f-Added t-Starting w-Stopped 
2023-03-17 14:54:24.265 30434-30592 o.o.a.ShareManager      org.onionshare.android.nightly       E  Unhandled state: ↑
2023-03-17 14:54:24.270 30434-30963 o.o.a.ShareManager      org.onionshare.android.nightly       I  New state from: f-Zipping t-Starting w-Stopped 
2023-03-17 14:54:24.270 30434-30963 o.o.a.ShareManager      org.onionshare.android.nightly       D  New state: Starting
---------------------------- PROCESS STARTED (30964) for package org.onionshare.android.nightly ----------------------------
2023-03-17 14:54:24.685 30434-30962 o.o.a.ShareManager      org.onionshare.android.nightly       I  New state from: f-Zipping t-Starting w-Stopped 
2023-03-17 14:54:24.686 30434-30962 o.o.a.ShareManager      org.onionshare.android.nightly       D  New state: Starting
2023-03-17 14:54:24.691 30434-30962 o.o.a.ShareManager      org.onionshare.android.nightly       I  New state from: f-Zipped t-Starting w-Stopped 
2023-03-17 14:54:24.694 30434-30962 k.application           org.onionshare.android.nightly       I  Autoreload is disabled because the development mode is off.
2023-03-17 14:54:24.701 30434-30962 k.application           org.onionshare.android.nightly       I  Application started in 0.008 seconds.
2023-03-17 14:54:24.701 30434-30962 k.application           org.onionshare.android.nightly       I  Application started: io.ktor.server.application.Application@ab08a0f
2023-03-17 14:54:24.704 30434-30962 TrafficStats            org.onionshare.android.nightly       D  tagSocket(156) with statsTag=0x42, statsUid=-1
2023-03-17 14:54:24.708 30434-30814 k.application           org.onionshare.android.nightly       I  Responding at http://0.0.0.0:17638
2023-03-17 14:54:24.709 30434-30962 o.o.a.ShareManager      org.onionshare.android.nightly       I  New state from: f-Zipped t-Starting w-Started 
```

Reply:
Hmm.. I couldn't reproduce this on an Android 8 phone.

>  "Unhandled state" is logged a few times - not sure if this is indicative of a problem?

No, we are not handling all possible combinations of state. Doesn't seem to be related to this issue.

>    The web server moves to the Stopped state before being restarted, and yet we get EADDRINUSE when trying to restart it

Yeah, in both cases I see `I  Application stopped: io.ktor.server.application.Application@cf1ac63` in the log, but only in yours the server didn't seem to free the address. 

As this issue most likely exists on main as well, maybe we create a ticket for it, but not let it block this MR?

Reply:
I did happen once to someone from Google Play: https://github.com/onionshare/onionshare-android/issues/82

Reply:
A separate ticket sounds good.

It occurred to me that it might not be an application bug, but might be caused by the kernel refusing to reuse a recently closed port. I couldn't remember the name for this, but a search turned this up:

https://unix.stackexchange.com/questions/17218/how-long-is-a-tcp-local-socket-address-that-has-been-bound-unavailable-after-clo

Perhaps different devices impose different delays?

In any case, it seems like an issue we should keep in mind (here and in other apps that reuse port numbers, like Briar's Tor and LAN plugins).